# translation of anaconda.master.ru.po to
# translation of ru.po to
# Leonid Kanter <leon@asplinux.ru>, 2004, 2006.
# Valery Suhomlinov <goodguy@goodguy.spb.ru>, 2004.
# Andrew Martynov <andrewm@inventa.ru>, 2004, 2005, 2006, 2008, 2009.
# Yuliya Poyarkova <ypoyarko@brisbane.redhat.com>, 2005.
# wedge <wedge@xwing>, 2006.
# Yuliya Poyarkova <ypoyarko@redhat.com>, 2006.
# Yulia Poyarkova <ypoyarko@redhat.com>, 2006.
# Nikolay Sivov <bunglehead@gmail.com>, 2007.
# Yulia Poyarkova <yulia.poyarkova@redhat.com>, 2008, 2009.
msgid ""
msgstr ""
"Project-Id-Version: anaconda.master.ru\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2009-08-05 10:18-0400\n"
"PO-Revision-Date: 2009-04-07 02:10+0400\n"
"Last-Translator: Andrew Martynov <andrewm@inventa.ru>\n"
"Language-Team: Russian <fedora-trans-ru@redhat.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Date: 1999-04-03 12:20+0200\n"
"X-Generator: Lokalize 0.3\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%"
"10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: anaconda:374
msgid "Press <enter> for a shell"
msgstr "Для перехода в командную оболочку нажмите <enter>"

#: anaconda:389 rescue.py:324 rescue.py:350 rescue.py:363 rescue.py:439
#: rescue.py:451 text.py:545 loader/cdinstall.c:228 loader/cdinstall.c:232
#: loader/cdinstall.c:248 loader/cdinstall.c:251 loader/cdinstall.c:445
#: loader/cdinstall.c:452 loader/cdinstall.c:517 loader/dirbrowser.c:145
#: loader/driverdisk.c:98 loader/driverdisk.c:243 loader/driverdisk.c:274
#: loader/driverdisk.c:305 loader/driverdisk.c:340 loader/driverdisk.c:359
#: loader/driverdisk.c:371 loader/driverdisk.c:379 loader/driverdisk.c:539
#: loader/driverdisk.c:576 loader/driverselect.c:81 loader/driverselect.c:205
#: loader/hdinstall.c:134 loader/hdinstall.c:280 loader/hdinstall.c:343
#: loader/hdinstall.c:375 loader/hdinstall.c:445 loader/hdinstall.c:488
#: loader/hdinstall.c:501 loader/kbd.c:138 loader/kickstart.c:132
#: loader/kickstart.c:142 loader/kickstart.c:185 loader/kickstart.c:284
#: loader/kickstart.c:375 loader/kickstart.c:554 loader/lang.c:115
#: loader/lang.c:379 loader/loader.c:403 loader/loader.c:439
#: loader/loader.c:482 loader/loader.c:503 loader/loader.c:520
#: loader/loader.c:557 loader/loader.c:1207 loader/loader.c:1382
#: loader/mediacheck.c:46 loader/mediacheck.c:85 loader/mediacheck.c:92
#: loader/mediacheck.c:101 loader/method.c:279 loader/method.c:353
#: loader/modules.c:335 loader/modules.c:347 loader/net.c:572 loader/net.c:943
#: loader/net.c:1577 loader/net.c:1598 loader/net.c:1856 loader/net.c:1878
#: loader/net.c:1890 loader/nfsinstall.c:91 loader/nfsinstall.c:248
#: loader/nfsinstall.c:267 loader/nfsinstall.c:335 loader/telnetd.c:92
#: loader/urlinstall.c:83 loader/urlinstall.c:107 loader/urlinstall.c:175
#: loader/urlinstall.c:418 loader/urlinstall.c:427 loader/urlinstall.c:436
#: loader/urls.c:292 loader/urls.c:328 loader/urls.c:334 loader/urls.c:346
#: textw/constants_text.py:44
msgid "OK"
msgstr "OК"

#: anaconda:396
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""
"Недостаточно ОЗУ для запуска графической установки. Установка будет "
"продолжена в текстовом режиме."

#: anaconda:459
#, fuzzy
msgid "Starting graphical installation."
msgstr "Запуск графической установки..."

#: anaconda:775
msgid "Would you like to use VNC?"
msgstr "Хотите использовать VNC?"

#: anaconda:776
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""
"Текстовый режим предоставляет ограниченный набор возможностей. Он не "
"позволяет задать собственное разбиение диска или сделатьвыбор набора "
"пакетов. Не желаете ли использовать режим VNC вместо него?"

#: anaconda:802
msgid "Install class forcing text mode installation"
msgstr "Переход к установке в текстовом режиме"

#: anaconda:821
#, fuzzy
msgid "Graphical installation is not available. Starting text mode."
msgstr "Графическая установка невозможна...  Запускается текстовая установка."

#: anaconda:829
#, fuzzy
msgid "DISPLAY variable not set. Starting text mode."
msgstr "Переменная DISPLAY не установлена. Запускается текстовый режим!"

#: anaconda:997
msgid "reipl configuration successful => reboot"
msgstr "настройка reipl прошла удачно => перезагрузка"

#: anaconda:1000
msgid "reipl configuration failed => halt"
msgstr "настройка reipl не удалась => останов"

#: backend.py:145
#, python-format
msgid "Upgrading %s\n"
msgstr "Обновление %s\n"

#: backend.py:147
#, python-format
msgid "Installing %s\n"
msgstr "Установка %s\n"

#: backend.py:167
msgid "Copying File"
msgstr "Копирование файла"

#: backend.py:168
#, fuzzy
msgid "Transferring install image to hard drive"
msgstr "Установочный образ переносится на жесткий диск..."

#: backend.py:178
#, fuzzy
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is often cause by damaged or low quality media."
msgstr ""
"Произошла ошибка при переносе установочного образа на жесткий диск. Наиболее "
"вероятная причина - испорченный носитель."

#: backend.py:182
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""
"Произошла ошибка при переносе установочного образа на жесткий диск. Наиболее "
"вероятная причина - отсутствие места на диске."

#: backend.py:186 image.py:285 livecd.py:192 livecd.py:396
#: partIntfHelpers.py:208 text.py:308 text.py:312 yuminstall.py:364
#: yuminstall.py:686 yuminstall.py:800 yuminstall.py:1081 yuminstall.py:1147
#: yuminstall.py:1304 yuminstall.py:1322 iw/autopart_type.py:99
#: iw/autopart_type.py:198 iw/autopart_type.py:318 iw/autopart_type.py:321
#: iw/autopart_type.py:388 iw/autopart_type.py:396 iw/autopart_type.py:425
#: iw/osbootwidget.py:210 iw/osbootwidget.py:219 iw/raid_dialog_gui.py:705
#: iw/raid_dialog_gui.py:744 iw/task_gui.py:63 iw/task_gui.py:161
#: iw/task_gui.py:292 iw/task_gui.py:414 loader/cdinstall.c:228
#: loader/cdinstall.c:517 loader/driverdisk.c:98 loader/driverdisk.c:305
#: loader/driverdisk.c:340 loader/driverdisk.c:371 loader/driverdisk.c:379
#: loader/driverdisk.c:443 loader/hdinstall.c:134 loader/hdinstall.c:343
#: loader/hdinstall.c:445 loader/hdinstall.c:488 loader/hdinstall.c:501
#: loader/kickstart.c:284 loader/lang.c:115 loader/loader.c:403
#: loader/loader.c:520 loader/loader.c:1207 loader/mediacheck.c:46
#: loader/mediacheck.c:85 loader/mediacheck.c:92 loader/method.c:279
#: loader/method.c:353 loader/nfsinstall.c:248 loader/nfsinstall.c:267
#: loader/telnetd.c:92 loader/urlinstall.c:83 loader/urlinstall.c:107
#: loader/urlinstall.c:175 loader/urls.c:328 loader/urls.c:334
#: storage/__init__.py:1577 storage/__init__.py:1593
#: textw/netconfig_text.py:272 textw/partition_text.py:186
#: textw/partition_text.py:192 textw/partition_text.py:198
#: textw/partition_text.py:231 textw/partition_text.py:241
#: textw/upgrade_text.py:175 textw/upgrade_text.py:182
msgid "Error"
msgstr "Ошибка"

#: bootloader.py:53 bootloader.py:209 bootloader.py:216 image.py:82
#: upgrade.py:242 yuminstall.py:1485 yuminstall.py:1521
#: iw/autopart_type.py:208 iw/blpasswidget.py:151 iw/upgrade_swap_gui.py:192
#: iw/upgrade_swap_gui.py:200 iw/upgrade_swap_gui.py:207
#: storage/devicetree.py:141 storage/devicetree.py:168
#: textw/upgrade_text.py:187
msgid "Warning"
msgstr "Внимание!"

#: bootloader.py:54
#, fuzzy
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"Ваши файловые системы уже активированы. Вы не можете вернуться назад.\n"
"\n"
"Продолжить установку?"

#: bootloader.py:58 gui.py:1093 gui.py:1140 gui.py:1268 gui.py:1345
#: image.py:91 livecd.py:199 livecd.py:403 packages.py:126 packages.py:134
#: packages.py:142 packages.py:150 packages.py:162 packages.py:171
#: packages.py:179 packages.py:188 upgrade.py:57 upgrade.py:197
#: yuminstall.py:804 yuminstall.py:884 yuminstall.py:890 yuminstall.py:1052
#: yuminstall.py:1077 yuminstall.py:1135 yuminstall.py:1299 yuminstall.py:1329
#: storage/__init__.py:89 storage/__init__.py:1679
msgid "_Exit installer"
msgstr "В_ыход из программы установки"

#: bootloader.py:58 image.py:92 image.py:248 kickstart.py:1301
#: kickstart.py:1340 upgrade.py:57 yuminstall.py:1140 iw/partition_gui.py:1036
#: storage/__init__.py:1680
msgid "_Continue"
msgstr "П_родолжить"

#: bootloader.py:114 platform.py:93 platform.py:319 platform.py:376
#: platform.py:466
msgid "RAID Device"
msgstr "устройство RAID"

#: bootloader.py:143
msgid "Bootloader"
msgstr "Загрузчик"

#: bootloader.py:143
#, fuzzy
msgid "Installing bootloader."
msgstr "Установка загрузчика..."

#: bootloader.py:210
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""

#: bootloader.py:217
#, fuzzy
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""
"В вашей системе не установлен ни один пакет с ядром Linux. Конфигурация "
"загрузчика не будет изменена."

#: cmdline.py:55
msgid "Completed"
msgstr "Выполнено"

#: cmdline.py:63
#, fuzzy
msgid "In progress"
msgstr "В процессе...   "

#: cmdline.py:86 gui.py:1135 text.py:391
#, fuzzy, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"При обработке kickstart конфигурации произошла следующая ошибка:\n"
"\n"
"%s"

#: cmdline.py:98
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""

#: cmdline.py:114 cmdline.py:121
msgid "Can't have a question in command line mode!"
msgstr "Невозможно задать вопрос в режиме командной строки!"

#: constants.py:74
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"Произошла исключительная ситуация. Скорее всего, это ошибка. Сохраните "
"полный текст исключения и создайте отчет об ошибке"

#: constants.py:80
msgid " with the provider of this software."
msgstr " производителю этого программного компонента."

#: constants.py:84
#, python-format
msgid " against anaconda at %s"
msgstr " относительно Anaconda (%s)"

#: gui.py:106
#, fuzzy
msgid "An error occurred saving screenshots to disk."
msgstr "При копировании снимков экрана произошла ошибка."

#: gui.py:117
msgid "Screenshots Copied"
msgstr "Снимки экрана скопированы"

#: gui.py:118
#, fuzzy
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"Снимки экрана были скопированы в каталог:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"Вы сможете просмотреть их после перезагрузки системы."

#: gui.py:161
msgid "Saving Screenshot"
msgstr "Сохранение снимка экрана"

#: gui.py:162
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "Снимок экрана с именем '%s' сохранен."

#: gui.py:165
msgid "Error Saving Screenshot"
msgstr "Ошибка сохранения снимка экрана"

#: gui.py:166
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""
"При сохранении снимка экрана произошла ошибка. Если это произошло при "
"установке пакетов, вам необходимо попробовать еще несколько раз для "
"получения снимка."

#: gui.py:560 text.py:348
msgid "Installation Key"
msgstr "Установочный ключ"

#: gui.py:631 text.py:148
#, fuzzy
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""
"Выберите пароль для зашифрованных устройств. Ввод пароля будет будет "
"запрошен при каждой загрузке системы."

#: gui.py:650 gui.py:658 text.py:186 text.py:196
msgid "Error with passphrase"
msgstr "Ошибка пароля"

#: gui.py:651 text.py:187
msgid "The passphrases you entered were different.  Please try again."
msgstr "Введенные вами пароли не совпадают.  Повторите ввод."

#: gui.py:659
msgid "The passphrase must be at least eight characters long."
msgstr "Пароль должен содержать не менее 8 символов."

#: gui.py:693 text.py:224
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""
"Устройство %s зашифровано. Чтобы получить доступ к содержимому устройства в "
"процессе установки, необходимо ввести пароль."

#: gui.py:784 gui.py:1345 partIntfHelpers.py:153 partIntfHelpers.py:310
#: text.py:99 text.py:100 iw/account_gui.py:135 loader/dirbrowser.c:145
#: loader/driverdisk.c:244 loader/kickstart.c:375 loader/loader.c:439
#: loader/loader.c:557 textw/constants_text.py:48
msgid "Cancel"
msgstr "Отмена"

#: gui.py:796 gui.py:797 gui.py:909 gui.py:910 ui/anaconda.glade:154
msgid "_Debug"
msgstr "_Отладка"

#: gui.py:1137 text.py:393
msgid "Error Parsing Kickstart Config"
msgstr "Ошибка при обработке конфигурации Kickstart"

#: gui.py:1177
msgid "default:LTR"
msgstr "default:LTR"

#: gui.py:1257 text.py:510
msgid "Error!"
msgstr "Ошибка!"

#: gui.py:1258 text.py:511
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"Ошибка при попытке загрузки компонента интерфейса установщика.\n"
"\n"
"className = %s"

#: gui.py:1263 image.py:164 image.py:194 packages.py:425 packages.py:430
msgid "_Exit"
msgstr "_Выход"

#: gui.py:1264 image.py:164 image.py:194 livecd.py:199 yuminstall.py:678
#: yuminstall.py:1077 yuminstall.py:1135 yuminstall.py:1299
msgid "_Retry"
msgstr "_Повторить"

#: gui.py:1267
#, fuzzy
msgid "The installer will now exit."
msgstr "Программа установки завершает свою работу..."

#: gui.py:1270 storage/partitioning.py:256
#, fuzzy
msgid "The system will now reboot."
msgstr "Система сейчас будет перезагружена."

#: gui.py:1271 image.py:248 packages.py:433 yuminstall.py:1345
msgid "_Reboot"
msgstr "_Перегрузить"

#: gui.py:1273
msgid "Exiting"
msgstr "Завершение работы"

#: gui.py:1342 livecd.py:115 text.py:266
msgid "Exit installer"
msgstr "Прервать установку"

#: gui.py:1343
#, fuzzy
msgid "Are you sure you wish to exit the installer?"
msgstr "Вы действительно хотите прервать установку?"

#: gui.py:1352
#, python-format
msgid "%s Installer"
msgstr "Программа установки %s"

#: gui.py:1358
msgid "Unable to load title bar"
msgstr "Невозможно загрузить заголовок"

#: gui.py:1420
msgid "Install Window"
msgstr "Окно установки"

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""
"Размер ISO образа %s не кратен 2048 байтам. Это может означать, что он был "
"поврежден при копировании на эту машину.\n"
"\n"
"Рекомендуется перезагрузить машину и прервать установку, но вы можете "
"продолжить, если считаете, что это не ошибка."

#: image.py:156
msgid "Couldn't Mount ISO Source"
msgstr "Невозможно смонтировать образ ISO"

#: image.py:157
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""
"Произошла ошибка при монтировании устройства %s. Это могло произойти из-за "
"того, что ваши ISO образы находятся на устройствах LVM или RAID или из-за "
"проблем с монтированием раздела. Нажмите кнопку выхода, чтобы прервать "
"установку."

#: image.py:185
msgid "Missing ISO 9660 Image"
msgstr "Отсутствует образ диска ISO 9660"

#: image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""
"Программа установки попыталась смонтировать образ #%s, но он не найден на "
"жестком диске.\n"
"\n"
"Скопируйте данный образ на накопитель и нажмите «Повторить». Нажмите «Выход», "
"чтобы прервать установку."

#: image.py:241
msgid "Required Install Media"
msgstr "Требуемый установочный носитель"

#: image.py:242
#, python-format
msgid ""
"The software you have selected to install will require the following %s %s "
"discs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""
"Для установки выбранных вами пакетов необходимы следующие диски %s %s:\n"
"\n"
"%s\n"
"Перед продолжением установки убедитесь, что имеются в наличии все "
"перечисленные диски. Для остановки процесса установки нажмите кнопку "
"\"Перезагрузка\"."

#: image.py:248 livecd.py:402 packages.py:379 packages.py:430 packages.py:433
#: upgrade.py:196 yuminstall.py:804 yuminstall.py:890 yuminstall.py:1329
#: yuminstall.py:1345
msgid "_Back"
msgstr "_Назад"

#: image.py:286
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"Произошла ошибка отключения CD. Убедитесь, что к %s не осуществляется "
"обращение из оболочки на tty2, и нажмите OК для повтора."

#: installclass.py:74
msgid "Install on System"
msgstr "Установка в системе"

#: iutil.py:755
#, python-format
msgid "Error: On open, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"Ошибка: При операции открытия: невозможно задать reIPL метод в %s (%s: %s)"

#: iutil.py:763
#, python-format
msgid "Error: On write, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"Ошибка: При операции записи: невозможно задать reIPL метод в %s (%s: %s)"

#: iutil.py:770
#, python-format
msgid "Error: On close, cannot set reIPL method to %s (%s: %s)"
msgstr ""
"Ошибка: При операции закрытия: невозможно задать reIPL метод в %s (%s: %s)"

#: iutil.py:787
#, python-format
msgid "Error: Could not set %s as reIPL device (%s)"
msgstr "Ошибка: Невозможно задать %s в качестве reIPL устройства (%s)"

#: iutil.py:796
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr "Ошибка: Невозможно сбросить loadparm (%s)"

#: iutil.py:805
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr "Предупреждение: Невозможно сбросить parm (%s)"

#: iutil.py:815
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""
"После выключения выполните загрузку IPL вручную с DASD устройства %s для "
"продолжения установки"

#: iutil.py:837
#, python-format
msgid "Error: reading FCP property %s for reIPL (%s)"
msgstr "Ошибка: чтение реквизита FCP  %s для reIPL (%s)"

#: iutil.py:850
#, python-format
msgid "Error: writing FCP property %s for reIPL (%s)"
msgstr "Ошибка: запись реквизита FCP  %s для reIPL (%s)"

#: iutil.py:863
#, python-format
msgid "Error: writing default FCP property %s for reIPL (%s)"
msgstr "Ошибка: запись реквизита по умолчанию FCP  %s для reIPL (%s)"

#: iutil.py:873
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN %"
"(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""
"После выключения выполните загрузку IPL вручную с FCP %(device)s c WWPN %"
"(wwpn)s и LUN %(lun)s для продолжения установки "

#: iutil.py:880
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""
"После выключения выполните загрузку IPL вручную с устройства содержащего "
"теперь /boot для продолжения установки"

#: iutil.py:888
msgid "Error determining mount point type"
msgstr "Ошибка определения типа точки монтирования"

#: iutil.py:892
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr ""
"Точка монтирования /boot или / расположена на диске, с которым нет "
"возможности работать"

#: kickstart.py:96
#, fuzzy, python-format
msgid ""
"There was an error running the kickstart script at line %s.  You may examine "
"the output in %s.  This is a fatal error and installation will be aborted.  "
"Press the OK button to exit the installer."
msgstr ""
"Произошла критическая ошибка при выполнении сценария кикстарта в строке %s. "
"Вы можете просмотреть вывод в %s. Работа программы установки будет прервана. "
"Нажмите OK для выхода из программы установки."

#: kickstart.py:111 kickstart.py:113
msgid "Scriptlet Failure"
msgstr "Сбой скриплета"

#: kickstart.py:1167
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr "Ошибка обработки строк %%ksappend: %s"

#: kickstart.py:1170
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr "Неизвестная ошибка при обработке строк %%ksappend: %s"

#: kickstart.py:1181
#, fuzzy, python-format
msgid ""
"The following error was found while parsing thekickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"При обработке kickstart конфигурации произошла следующая ошибка:\n"
"\n"
"%s"

#: kickstart.py:1242 livecd.py:223
#, fuzzy
msgid "Post-Installation"
msgstr "После установки"

#: kickstart.py:1243
#, fuzzy
msgid "Running post-installation scripts"
msgstr "Выполняются пост-установочные сценарии"

#: kickstart.py:1259
#, fuzzy
msgid "Pre-Installation"
msgstr "Установка пакета"

#: kickstart.py:1260
#, fuzzy
msgid "Running pre-installation scripts"
msgstr "Выполняются пред-установочные сценарии"

#: kickstart.py:1292
msgid "Missing Package"
msgstr "Отсутствующий пакет"

#: kickstart.py:1293
#, fuzzy, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""
"Вы указали, что пакет '%s' должен быть установлен.  Этот пакет не "
"существует. Продолжить или прервать установку?"

#: kickstart.py:1299 kickstart.py:1338
msgid "_Abort"
msgstr "_Прервать"

#: kickstart.py:1300 kickstart.py:1339
msgid "_Ignore All"
msgstr "_Игнорировать все"

#: kickstart.py:1330
msgid "Missing Group"
msgstr "Отсутствует группа"

#: kickstart.py:1331
#, fuzzy, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""
"Вы указали, что группа '%s' должна быть установлена.  Эта группа не "
"существует. Продолжить или прервать установку?"

#: kickstart.py:1437
#, fuzzy, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""
"В вашем kickstart файле отсутствует требуемая информация, которую anaconda "
"не может запросить. Добавьте в файл следующий раздел и повторите попытку "
"снова:\n"
"%s"

#: livecd.py:110
msgid "Unable to find image"
msgstr "Не удается найти образ"

#: livecd.py:111
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr "Указанный путь не является корректным %s live CD для установки."

#: livecd.py:174
msgid "Copying live image to hard drive."
msgstr "Установочный live образ переносится на жесткий диск."

#: livecd.py:193
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""
"Ошибка при установке «живого» образа на жесткий диск. Возможная причина -- "
"сбойный носитель.\n"
"Проверьте целостность установочного носителя.\n"
"\n"
"При отмене установки система останется в нестабильном состоянии, требующим "
"вероятной переустановки."

#: livecd.py:224
#, fuzzy
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""
"Запуск заключительные операции с фаловой системой.  Это может занять "
"несколько минут..."

#: livecd.py:397
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""
"Созданная вами корневая файловая система слишком мала для размещения образа "
"(требуется %.2f Мб)."

#: network.py:56
msgid "Hostname must be 255 or fewer characters in length."
msgstr "Имя узла не может содержать более 255 символов."

#: network.py:62
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or '0-"
"9'"
msgstr "Имя узла может начинаться с символов a-z, A-Z, 0-9."

#: network.py:67
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr "Имя узла может содержать символы a-z, A-Z, 0-9, дефис и точку."

#: network.py:176
msgid "IP address is missing."
msgstr "Адрес IP не указан."

#: network.py:180
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""
"Адрес IP должен включать четыре числа в диапазоне от 0 до 255, разделённых "
"точками."

#: network.py:183
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "Недопустимый адрес IPv6: '%s'"

#: network.py:185
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "Недопустимый адрес IP: '%s'."

#: packages.py:121
msgid "Device Resize Failed"
msgstr "Сбой операции изменения размера устройства"

#: packages.py:122
#, python-format
msgid "An error was encountered while resizing device %s."
msgstr "Произошла ошибка при изменении размера устройства %s."

#: packages.py:129
msgid "Device Creation Failed"
msgstr "Сбой операции создания устройства"

#: packages.py:130
#, python-format
msgid "An error was encountered while creating device %s."
msgstr "Произошла ошибка при создании устройства %s."

#: packages.py:137
msgid "Device Removal Failed"
msgstr "Сбой операции удаления устройства"

#: packages.py:138
#, python-format
msgid "An error was encountered while removing device %s."
msgstr "Произошла ошибка при удалении устройства %s."

#: packages.py:145
msgid "Device Setup Failed"
msgstr "Сбой операции настройки устройства"

#: packages.py:146
#, python-format
msgid "An error was encountered while setting up device %s."
msgstr "Произошла ошибка при настройке устройства %s."

#: packages.py:157
msgid "Resizing Failed"
msgstr "Сбой операции изменения размера"

#: packages.py:158
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr "Произошла ошибка при изменении размера устройства %s."

#: packages.py:165
msgid "Migration Failed"
msgstr "Сбой операции миграции"

#: packages.py:166
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr "Произошла ошибка при миграции файловой системы на устройстве %s."

#: packages.py:174
msgid "Formatting Failed"
msgstr "Сбой операции форматирования"

#: packages.py:175
#, python-format
msgid "An error was encountered while formatting device %s."
msgstr "Произошла ошибка при форматировании устройства %s."

#: packages.py:183
msgid "Storage Activation Failed"
msgstr "Сбой операции активизации хранилища"

#: packages.py:184
msgid "An error was encountered while activating your storage configuration."
msgstr "Произошла ошибка при активизации настроек вашего хранилища."

#: packages.py:188
msgid "_File Bug"
msgstr ""

#: packages.py:350
msgid "Invalid Key"
msgstr "Неверный ключ"

#: packages.py:351
msgid "The key you entered is invalid."
msgstr "Введенный ключ неверен."

#: packages.py:376 rescue.py:287 rescue.py:289 loader/cdinstall.c:248
#: loader/cdinstall.c:251 loader/method.c:326 textw/upgrade_text.py:133
msgid "Skip"
msgstr "Пропустить"

#: packages.py:379
msgid "_Skip"
msgstr "_Пропустить"

#: packages.py:411 packages.py:434
msgid "Warning! This is pre-release software!"
msgstr "Внимание! Это предварительный выпуск!"

#: packages.py:412
#, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""
"Спасибо за загрузку этого предварительного выпуска %s.\n"
"\n"
"Этот выпуск не является окончательным и не предназначен для повседневного и "
"промышленного использования. Цель этого выпуска - сбор отчетов от "
"испытателей.\n"
"\n"
"Чтобы сообщить о найденных проблемах, посетите:\n"
"\n"
"   %s\n"
"\n"
"и заполните отчет относительно '%s'.\n"

#: packages.py:425
msgid "_Install anyway"
msgstr "_Установить всё равно"

#: packages.py:429
msgid "The installer will now exit..."
msgstr "Программа установки завершает свою работу..."

#: packages.py:432
msgid "Your system will now be rebooted..."
msgstr "Ваша система сейчас будет перезагружена..."

#: partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr "Введите имя группы томов."

#: partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr "Имя группы томов должно быть меньше 128 символов"

#: partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "Ошибка - имя группы томов %s недействительно."

#: partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Ошибка - имя группы томов содержит недопустимые символы или пробелы.  "
"Допустимы буквы, цифры, '.' или '_'."

#: partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr "Введите имя логического тома."

#: partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr "Имена логических томов должны быть менее 128 символов"

#: partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "Ошибка - имя логического тома %s недействительно."

#: partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Ошибка - имя логического тома содержит недопустимые символы или пробелы.  "
"Допустимы буквы, цифры, '.' или '_'."

#: partIntfHelpers.py:101
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""
"Недопустимая точка монтирования %s.  Точки монтирования должны начинаться с "
"'/', не могут заканчиваться на '/' и могут содержать только печатные символы "
"без пробелов."

#: partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr "Укажите точку монтирования для этого раздела."

#: partIntfHelpers.py:121 partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr "Невозможно удалить"

#: partIntfHelpers.py:122
msgid "You must first select a partition to delete."
msgstr "Необходимо указать раздел для удаления."

#: partIntfHelpers.py:149 partIntfHelpers.py:309 iw/lvm_dialog_gui.py:812
msgid "Confirm Delete"
msgstr "Подтвердите удаление"

#: partIntfHelpers.py:150
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "Вы собираетесь удалить все разделы на устройстве '%s'."

#: partIntfHelpers.py:153 partIntfHelpers.py:310 iw/lvm_dialog_gui.py:815
#: iw/lvm_dialog_gui.py:1348 iw/osbootwidget.py:103 iw/partition_gui.py:1336
msgid "_Delete"
msgstr "_Удалить"

#: partIntfHelpers.py:175
msgid "Notice"
msgstr "Примечание"

#: partIntfHelpers.py:176
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"Следующие разделы не были удалены, потому что они используются:\n"
"\n"
"%s"

#: partIntfHelpers.py:191
msgid "Format as Swap?"
msgstr "Форматировать как Swap?"

#: partIntfHelpers.py:192
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"%s имеет тип 0x82 (Linux swap), но он не отформатирован как раздел подкачки "
"Linux. \n"
"Отформатировать этот раздел как раздел подкачки?"

#: partIntfHelpers.py:207
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr "Необходимо выбрать хотя бы один жесткий диск для установки %s."

#: partIntfHelpers.py:212
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""
"Вы решили использовать для этой установки ранее существовавший раздел без "
"его форматирования. Мы рекомендуем отформатировать этот раздел, чтобы файлы "
"от предыдущей установки не вызвали проблем с этой установкой Linux. Если же "
"раздел содержит файлы, которые необходимо сохранить, такие как домашние "
"каталоги пользователей, вы можете продолжить без форматирования этого "
"раздела."

#: partIntfHelpers.py:220
msgid "Format?"
msgstr "Форматировать?"

#: partIntfHelpers.py:220 iw/partition_gui.py:1034
msgid "_Modify Partition"
msgstr "_Изменить раздел"

#: partIntfHelpers.py:220
msgid "Do _Not Format"
msgstr "Не форматировать"

#: partIntfHelpers.py:228
msgid "Error with Partitioning"
msgstr "Ошибка разбиения"

#: partIntfHelpers.py:229
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""
"Запрошенная вами схема разбиения вызвала следующие критические ошибки. Эти "
"ошибки должны быть исправлены для продолжения установки %s.\n"
"\n"
"%s"

#: partIntfHelpers.py:243
msgid "Partitioning Warning"
msgstr "Предостережение разбиения"

#: partIntfHelpers.py:244
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"Выбранная вами схема разбиения вызвала следующие предостережения.\n"
"\n"
"%s\n"
"\n"
"Вы действительно хотите продолжить с выбранной схемой разбиения?"

#: partIntfHelpers.py:258 iw/partition_gui.py:639
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"Следующие ранее существовавшие разделы будут отформатированы с удалением "
"всех данных."

#: partIntfHelpers.py:261
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"Нажмите 'Да' для продолжения и форматирования этих разделов. Нажатие кнопки "
"'Нет' позволит вернуться и изменить эти установки."

#: partIntfHelpers.py:267
msgid "Format Warning"
msgstr "Предостережение форматирования"

#: partIntfHelpers.py:293
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"Вы собираетесь удалить группу томов \"%s\"\n"
"\n"
"Все логические тома в этой группе будут утеряны!"

#: partIntfHelpers.py:297
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "Вы собираетесь удалить логический том \"%s\"."

#: partIntfHelpers.py:300
msgid "You are about to delete a RAID device."
msgstr "Вы собираетесь удалить устройство RAID."

#: partIntfHelpers.py:302
#, python-format
msgid "You are about to delete the %s partition."
msgstr "Вы собираетесь удалить раздел %s."

#: partIntfHelpers.py:306
#, python-format
msgid "You are about to delete the %s %s"
msgstr "Вы собираетесь удалить %s %s"

#: partIntfHelpers.py:317
msgid "Confirm Reset"
msgstr "Подтвердите возврат"

#: partIntfHelpers.py:318
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""
"Вы уверены, что хотите вернуть первоначальное состояние таблицы разделов?"

#: platform.py:94 platform.py:97 platform.py:320 platform.py:377
#: platform.py:467 platform.py:470
msgid "Master Boot Record (MBR)"
msgstr "Основная загрузочная запись (MBR)"

#: platform.py:96 platform.py:469
msgid "First sector of boot partition"
msgstr "Первый сектор загрузочного раздела"

#: platform.py:112
#, fuzzy
msgid "You have not created a bootable partition."
msgstr "Вы не создали загрузочный раздел."

#: platform.py:115
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "Загружаемые разделы допускаются только на устройствах RAID1."

#: platform.py:119
msgid "Bootable partitions cannot be on a logical volume."
msgstr "Загружаемые разделы на логических томах не допускаются"

#: platform.py:123
msgid "Bootable partitions cannot be on a RAID device."
msgstr "Загружаемые разделы не могут располагаться на устройствах RAID."

#: platform.py:127 platform.py:131
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "Загружаемые разделы не допускаются в файловой системе %s."

#: platform.py:135 platform.py:140
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""
"Загружаемые разделы не допускаются на зашифрованных блочных устройствах."

#: platform.py:213
msgid "EFI System Partition"
msgstr "Системный раздел EFI"

#: platform.py:218
#, fuzzy
msgid "You have not created a /boot/efi partition."
msgstr "Вы не создали загрузочный раздел."

#: platform.py:224
#, fuzzy
msgid "/boot is not on an ext2 filesystem."
msgstr "Загружаемые разделы не допускаются в файловой системе %s."

#: platform.py:227
msgid "/boot/efi is not EFI."
msgstr ""

#: platform.py:265
#, python-format
msgid "%s must have a bsd disk label."
msgstr ""

#: platform.py:277
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr ""

#: platform.py:322
msgid "PPC PReP Boot"
msgstr "PPC PReP Boot"

#: platform.py:334
#, fuzzy
msgid "The boot partition must be within the first 4MB of the disk."
msgstr "Проблема при записи состояния системы на диск."

#: platform.py:379 platform.py:382
msgid "Apple Bootstrap"
msgstr "Apple Bootstrap"

#: platform.py:397
#, python-format
msgid "%s must have a mac disk label."
msgstr ""

#: rescue.py:199
msgid "When finished please exit from the shell and your system will reboot."
msgstr "После завершения выйдите из оболочки и система будет перезагружена."

#: rescue.py:213
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "Не удается найти /bin/sh для запуска. Оболочка не будет запущена."

#: rescue.py:232
msgid "Setup Networking"
msgstr "Настройка сети"

#: rescue.py:233
msgid "Do you want to start the network interfaces on this system?"
msgstr "Активировать сетевые интерфейсы этой системы?"

#: rescue.py:234 loader/driverdisk.c:483 loader/driverdisk.c:493
#: loader/hdinstall.c:222 textw/constants_text.py:56
msgid "Yes"
msgstr "Да"

#: rescue.py:234 rescue.py:236 loader/driverdisk.c:483 loader/driverdisk.c:493
#: textw/constants_text.py:60
msgid "No"
msgstr "Нет"

#: rescue.py:240 yuminstall.py:1047
msgid "No Network Available"
msgstr "Сеть недоступна"

#: rescue.py:241
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""
"Невозможно активировать сетевое устройство. Сетевое окружение не будет "
"доступно в режиме восстановления."

#: rescue.py:276 rescue.py:345 rescue.py:356 rescue.py:434
msgid "Rescue"
msgstr "Восстановить"

#: rescue.py:277
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"В режиме восстановления будет выполнена попытка найти вашу установку Linux и "
"смонтировать ее под каталогом %s.  После этого вы сможете внести в систему "
"необходимые изменения. Нажмите клавишу \"Продолжить\". Вы также можете "
"смонтировать ваши файловые системы в режиме \"только для чтения\" вместо "
"\"чтение-запись\" нажав кнопку \"Только для чтения\".\n"
"\n"
"Если по некоторым причинам происходит сбой этого процесса, нажмите кнопку "
"\"Пропустить\" для быстрого перехода в командную оболочку.\n"
"\n"

#: rescue.py:287 iw/partition_gui.py:543 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/driverdisk.c:444 storage/__init__.py:134
#: storage/devicetree.py:82
msgid "Continue"
msgstr "Продолжить"

#: rescue.py:287 rescue.py:292
msgid "Read-Only"
msgstr "Только для чтения"

#: rescue.py:321
msgid "System to Rescue"
msgstr "Система для восстановления"

#: rescue.py:322
msgid "Which device holds the root partition of your installation?"
msgstr "Укажите устройство содержащее корневой раздел вашей установки."

#: rescue.py:324 rescue.py:328 text.py:515 text.py:517
msgid "Exit"
msgstr "Выход"

#: rescue.py:346
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""
"В вашей системе имеются \"грязные\" файловые системы, которые не были "
"выбраны для монтирования. Нажмите \"Enter\" для входа в оболочку, где можно "
"будет выполнить fsck и смонтировать эти разделы. После выхода из оболочки "
"система будет автоматически перезагружена."

#: rescue.py:357
#, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""
"Система смонтирована под %s.\n"
"\n"
"Нажмите <Enter> для входа в оболочку. Если вы хотите сделать вашу систему "
"корневым окружением, выполните команду:\n"
"\n"
"\tchroot %s\n"
"\n"
"Система будет перезагружена автоматически после выхода из оболочки."

#: rescue.py:435
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""
"При монтировании системы или ее части произошла ошибка. Часть её могла быть "
"смонтирована под %s.\n"
"\n"
"Нажмите <Enter> для входа в оболочку. Система будет перезагружена "
"автоматически после выхода из оболочки."

#: rescue.py:444
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: rescue.py:447
msgid "Rescue Mode"
msgstr "Режим восстановления"

#: rescue.py:448
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""
"Отсутствуют разделы Linux. Нажмите Enter для возврата в оболочку. Система "
"будет перезагружена автоматически после выхода из оболочки."

#: rescue.py:461
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "Ваша система смонтирована под каталогом %s."

#: text.py:153
msgid "Passphrase for encrypted device"
msgstr "Пароль для зашифрованного устройства"

#: text.py:166
msgid "Also add this passphrase to all existing encrypted devices"
msgstr "Используйте этот пароль для всех существующих зашифрованных устройств."

#: text.py:231 ui/lukspassphrase.glade:275
msgid "Passphrase"
msgstr "Пароль"

#: text.py:239 ui/lukspassphrase.glade:421
msgid "This is a global passphrase"
msgstr "Это глобальный пароль"

#: text.py:309 text.py:313
msgid "Repository editing is not available in text mode."
msgstr "Изменение репозитария невозможно в текстовом режиме."

#: text.py:353
#, python-format
msgid "Please enter your %(instkey)s"
msgstr "Введите свой %(instkey)s"

#: text.py:370 ui/instkey.glade:143
#, python-format
msgid "Skip entering %(instkey)s"
msgstr "Пропустить ввод %(instkey)s"

#: text.py:420 loader/lang.c:63 loader/loader.c:215
#, c-format, python-format
msgid "Welcome to %s for %s"
msgstr "Добро пожаловать в %s для %s"

#: text.py:422
#, python-format
msgid "Welcome to %s"
msgstr "Добро пожаловать в %s"

#: text.py:424
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""
" <Tab>/<Alt-Tab> между элементами | <Space> выбор | <F12> следующий экран "

#: text.py:515 loader/net.c:111 loader/net.c:462 loader/net.c:518
#: loader/net.c:661 loader/net.c:669 loader/net.c:1092 loader/net.c:1099
msgid "Retry"
msgstr "Повторить"

#: text.py:541
msgid "Cancelled"
msgstr "Отменено"

#: text.py:542
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""
"С этого места невозможно вернуться к предыдущему этапу. Вам придется "
"повторить попытку."

#: upgrade.py:50
msgid "Proceed with upgrade?"
msgstr "Продолжить обновление?"

#: upgrade.py:51
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"Файловые системы текущей установки Linux, выбранной для обновления, уже "
"смонтированы. Вернуться назад от этой точки невозможно.\n"
"\n"

#: upgrade.py:55
msgid "Would you like to continue with the upgrade?"
msgstr "Продолжить обновление?"

#: upgrade.py:183
msgid "Mount failed"
msgstr "Ошибка монтирования"

#: upgrade.py:184
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""
"Произошла ошибка при подключении файловых систем, указанных в файле /etc/"
"fstab вашей системы. Устраните эту проблему и повторите попытку "
"обновления. \n"
"%s"

#: upgrade.py:191
msgid "Upgrade root not found"
msgstr "Корневой каталог для обновления не обнаружен"

#: upgrade.py:192
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""
"Корневой каталог от предыдущей установленной системы не обнаружен. Вы можете "
"выйти из программы установки или сделать шаг назад и выбрать установку "
"вместо обновления."

#: upgrade.py:215
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""
"Следующие файлы являются абсолютными символическими ссылками, которые не "
"поддерживаются во время обновления. Замените их на относительные и повторите "
"попытку обновления.\n"

#: upgrade.py:221
msgid "Absolute Symlinks"
msgstr "Абсолютные символические ссылки"

#: upgrade.py:232
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"Перечисленные каталоги, которые должны быть символическими ссылками, могут "
"вызвать проблемы при обновлении. Восстановите их первоначальное состояние и "
"перезапустите обновление.\n"
"\n"

#: upgrade.py:238
msgid "Invalid Directories"
msgstr "Неверные каталоги"

#: upgrade.py:243
#, python-format
msgid "%s not found"
msgstr "%s не найдено"

#: vnc.py:137
#, python-format
msgid "%s %s installation on host %s"
msgstr "Установка %s %s на узле %s"

#: vnc.py:139
#, python-format
msgid "%s %s installation"
msgstr "Установка %s %s"

#: vnc.py:166
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Попытка соединения с клиентом vnc на узле %s..."

#: vnc.py:180
msgid "Connected!"
msgstr "Подключен!"

#: vnc.py:183
msgid "Will try to connect again in 15 seconds..."
msgstr "Попытка повторного соединения через 15 секунд..."

#: vnc.py:200
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "Подключите вручную ваш клиент vnc к %s для начала установки."

#: vnc.py:202
msgid "Please manually connect your vnc client to begin the install."
msgstr "Подключите ваш клиент vnc вручную для начала установки."

#: vnc.py:205
msgid "Starting VNC..."
msgstr "Запуск VNC..."

#: vnc.py:230
msgid "The VNC server is now running."
msgstr "Сервер VNC запущен."

#: vnc.py:243
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Вы выбрали подключение к прослушивающему vncviewer. \n"
"Для этого нет необходимости в указании пароля.\n"
"Если же вы определите пароль, он будет использоваться\n"
"в случае неудачи при попытке подключения к vncviewer.\n"
"\n"

#: vnc.py:248
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"Внимание!!! Сервер VNC запущен БЕЗ ПАРОЛЯ!\n"
"Используйте параметр загрузки vncpassword=<password>,\n"
"если хотите обезопасить сервер.\n"
"\n"

#: vnc.py:252
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Вы выбрали запуск vnc с паролем.\n"
"\n"

#: vnc.py:254
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Неизвестная ошибка. Остановка работы.\n"
"\n"

#: vnc.py:276 vnc.py:369
msgid "VNC Configuration"
msgstr "Настройка VNC"

#: vnc.py:279 vnc.py:373
msgid "No password"
msgstr "Без пароля"

#: vnc.py:281 vnc.py:376
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""
"При помощи пароля вы предотвратите несанкционированное подключение и "
"отслеживание процесса установки. Введите пароль, который будет использован "
"во время установки"

#: vnc.py:289 vnc.py:384 textw/userauth_text.py:47
msgid "Password:"
msgstr "Пароль:"

#: vnc.py:290 vnc.py:385 textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "Подтвердите пароль:"

#: vnc.py:308 vnc.py:407 textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "Пароли не совпадают"

#: vnc.py:309 vnc.py:408 textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr "Введенные вами пароли не совпадают. Повторите ввод."

#: vnc.py:314 vnc.py:413 textw/userauth_text.py:66
msgid "Password Length"
msgstr "Длина пароля"

#: vnc.py:315 vnc.py:414
msgid "The password must be at least six characters long."
msgstr "Пароль должен содержать не менее 6 символов."

#: vnc.py:337
msgid "Unable to Start X"
msgstr "Невозможно запустить X"

#: vnc.py:339
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"Невозможно запустить X (графическую подсистему) на вашей машине. Запустить "
"VNC для подключения к данному компьютеру с другой машины и выполнения "
"установки в графическом режиме или продолжить установку в текстовом режиме?"

#: vnc.py:358
msgid "Start VNC"
msgstr "Запустить VNC"

#: vnc.py:359 vnc.py:361
msgid "Use text mode"
msgstr "Использовать текстовый режим"

#: yuminstall.py:80
#, python-format
msgid "%s MB"
msgstr "%s Мб"

#: yuminstall.py:83
#, python-format
msgid "%s KB"
msgstr "%s Кб"

#: yuminstall.py:129
msgid "Preparing to install"
msgstr ""

#: yuminstall.py:130
#, fuzzy
msgid "Preparing transaction from installation source"
msgstr "Подготовка транзакции из источника установки..."

#: yuminstall.py:158
#, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "<b>Установка %s</b> (%s)\n"

#: yuminstall.py:218
msgid "Finishing upgrade"
msgstr "Завершение процесса обновления"

#: yuminstall.py:219
#, fuzzy
msgid "Finishing upgrade process.  This may take a little while."
msgstr "Запуск процесса обновления.  Это может занять несколько минут..."

#: yuminstall.py:286 iw/task_gui.py:301
msgid "Error Setting Up Repository"
msgstr "Ошибка настройки репозитория"

#: yuminstall.py:287 iw/task_gui.py:302
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%s\n"
"\n"
"Please provide the correct information for installing %s."
msgstr ""
"Ошибка при настройке установочного репозитория:\n"
"\n"
"%s\n"
"\n"
"Предоставьте корректную информацию для установки %s."

#: yuminstall.py:346
msgid "Change Disc"
msgstr "Замените диск"

#: yuminstall.py:347
#, python-format
msgid "Please insert %s disc %d to continue."
msgstr "Вставьте %s диск %d для продолжения."

#: yuminstall.py:357
msgid "Wrong Disc"
msgstr "Неверный диск"

#: yuminstall.py:358
#, python-format
msgid "That's not the correct %s disc."
msgstr "Неверный диск %s."

#: yuminstall.py:365
msgid "Unable to access the disc."
msgstr "Не удаётся получить доступ к CD-ROM."

#: yuminstall.py:524
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""
"В конфигурации отсутствует имя репозитория %r. Вместо имени будет "
"использоваться идентификатор."

#: yuminstall.py:676 yuminstall.py:678
msgid "Re_boot"
msgstr "_Перезагрузка"

#: yuminstall.py:676
msgid "_Eject"
msgstr "_Извлечь CD"

#: yuminstall.py:687
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""
"Не удается открыть файл %s. Возможная причина - отсутствие файла или сбойный "
"пакет/ носитель. Убедитесь, что дерево установки содержит все необходимые "
"пакеты.\n"
"\n"
"При отмене установки система останется в нестабильном состоянии, требующим "
"вероятной переустановки.\n"
"\n"

#: yuminstall.py:735
msgid "Retrying"
msgstr "Повторная попытка"

#: yuminstall.py:735
#, fuzzy
msgid "Retrying download."
msgstr "Повтор попытки получения..."

#: yuminstall.py:801
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""
"При выполнении вашей транзакции произошла ошибка по следующим причинам: %s\n"

#: yuminstall.py:836 yuminstall.py:837
msgid "file conflicts"
msgstr "конфликты файлов"

#: yuminstall.py:838
msgid "older package(s)"
msgstr "старые пакеты"

#: yuminstall.py:839
msgid "insufficient disk space"
msgstr "недостаточно места на диске"

#: yuminstall.py:840
msgid "insufficient disk inodes"
msgstr "недостаточное кол-во inode на диске"

#: yuminstall.py:841
msgid "package conflicts"
msgstr "конфликты пакетов"

#: yuminstall.py:842
msgid "package already installed"
msgstr "пакет уже установлен"

#: yuminstall.py:843
msgid "required package"
msgstr "требуется пакет"

#: yuminstall.py:844
msgid "package for incorrect arch"
msgstr "пакет с неверной архитектурой"

#: yuminstall.py:845
msgid "package for incorrect os"
msgstr "пакет с неверной операционной системой"

#: yuminstall.py:859
msgid "You need more space on the following file systems:\n"
msgstr "Вам требуется больше места на следующих файловых системах:\n"

#: yuminstall.py:872
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"При проверке пакетов для установки обнаружены конфликты файлов:\n"
"%s\n"

#: yuminstall.py:875
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): %"
"s.\n"
msgstr ""
"При выполнении вашей транзакции произошла ошибка по следующим причинам: %s\n"

#: yuminstall.py:882 yuminstall.py:887
msgid "Error Running Transaction"
msgstr "Ошибка при выполнении транзакции"

#: yuminstall.py:1048
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""
"Некоторые из ваших репозитариев требуют наличия подключения к сети, но "
"произошла ошибка при ее настройке для вашей системы."

#: yuminstall.py:1082
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""
"Невозможно прочесть информацию о группах из репозитариев. Это проблема "
"относится к генерированию установочного дерева."

#: yuminstall.py:1118
#, fuzzy
msgid "Retrieving installation information."
msgstr "Получение установочной информации..."

#: yuminstall.py:1120
#, fuzzy, python-format
msgid "Retrieving installation information for %s."
msgstr "Получение установочной информации для %s..."

#: yuminstall.py:1122
msgid "Installation Progress"
msgstr "Прогресс установки"

#: yuminstall.py:1135 textw/constants_text.py:64
msgid "Edit"
msgstr "Редактировать"

#: yuminstall.py:1148 yuminstall.py:1305
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"Не удаётся прочитать метаданные пакета. Возможная причина - отсутствие "
"каталога repodata.  Убедитесь, что дерево установки сформировано "
"правильно.  \n"
"\n"
"%s"

#: yuminstall.py:1323
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""
"Выбранные вами пакеты требуют для установки %d Мб свободного места, но у вас "
"нет достаточного объема.  Вы можете изменить ваш выбор или выйти из "
"программы установки."

#: yuminstall.py:1342
msgid "Reboot?"
msgstr "Перегрузить?"

#: yuminstall.py:1343
msgid "The system will be rebooted now."
msgstr "Система сейчас будет перезагружена."

#: yuminstall.py:1486
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"Похоже, что вы выполняете обновление системы, которая слишком устарела для "
"того, чтобы быть обновленной до этой версии %s. Вы действительно хотите "
"продолжить обновление?"

#: yuminstall.py:1522
#, python-format
msgid ""
"The arch of the release of %s you are upgrading to appears to be %s which "
"does not match your previously installed arch of %s.  This is likely to not "
"succeed.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"Вы выполняете обновление системы до версии %s для архитектуры %s, которая не "
"соответствует установленной архитектуре %s. Вы действительно хотите "
"продолжить обновление?"

#: yuminstall.py:1573
msgid "Post Upgrade"
msgstr "После обновления"

#: yuminstall.py:1574
#, fuzzy
msgid "Performing post-upgrade configuration"
msgstr "Выполняется настройка после обновления..."

#: yuminstall.py:1576
#, fuzzy
msgid "Post Installation"
msgstr "После установки"

#: yuminstall.py:1577
#, fuzzy
msgid "Performing post-installation configuration"
msgstr "Выполняется настройка после установки..."

#: yuminstall.py:1803
#, fuzzy
msgid "Installation Starting"
msgstr "Запуск установки"

#: yuminstall.py:1804
#, fuzzy
msgid "Starting installation process"
msgstr "Прогресс установки"

#: yuminstall.py:1842
msgid "Dependency Check"
msgstr "Проверка зависимостей"

#: yuminstall.py:1843
#, fuzzy
msgid "Checking dependencies in packages selected for installation"
msgstr "Проверяются зависимости выбранных для установки пакетов..."

#: installclasses/fedora.py:39
msgid "_Fedora"
msgstr "_Fedora"

#: installclasses/fedora.py:40 installclasses/rhel.py:45
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""
"Установка по умолчанию %s включает набор программ, подходящих для решения "
"обычных задач работы с Интернетом.  Поддержку каких дополнительных задач вы "
"хотели бы включить?"

#: installclasses/fedora.py:49
msgid "Office and Productivity"
msgstr "Офисные приложения"

#: installclasses/fedora.py:50 installclasses/rhel.py:57
#: installclasses/rhel.py:62
msgid "Software Development"
msgstr "Разработка ПО"

#: installclasses/fedora.py:51
msgid "Web Server"
msgstr "Веб-сервер"

#: installclasses/rhel.py:44
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: installclasses/rhel.py:54
msgid "Office"
msgstr "Офис"

#: installclasses/rhel.py:55
msgid "Multimedia"
msgstr "Мультимедиа"

#: installclasses/rhel.py:61
msgid "Web server"
msgstr "Веб-сервер"

#: installclasses/rhel.py:66
msgid "Virtualization"
msgstr "Виртуализация"

#: installclasses/rhel.py:67
msgid "Clustering"
msgstr "Кластеризация"

#: installclasses/rhel.py:68
msgid "Storage Clustering"
msgstr "Кластеризация устройств хранения"

#: installclasses/rhel.py:72
msgid "Installation Number"
msgstr "Номер установки"

#: installclasses/rhel.py:73
msgid ""
"Would you like to enter an Installation Number (sometimes called "
"Subscription Number) now? This feature enables the installer to access any "
"extra components included with your subscription.  If you skip this step, "
"additional components can be installed manually later.\n"
"\n"
"See http://www.redhat.com/InstNum/ for more information."
msgstr ""

#: installclasses/rhel.py:79
msgid ""
"If you cannot locate the Installation Number, consult http://www.redhat.com/"
"InstNum/"
msgstr ""

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr "Пакеты в %s"

#: iw/GroupSelector.py:424
#, python-format
msgid "Optional packages selected: %d of %d"
msgstr "Выбраны дополнительные пакеты: %d из %d"

#: iw/GroupSelector.py:425
#, python-format
msgid "<i>%s</i>"
msgstr "<i>%s</i>"

#: iw/GroupSelector.py:485
msgid "Uncategorized"
msgstr "Вне категорий"

#: iw/account_gui.py:52
msgid "Root _Password:"
msgstr "_Пароль пользователя root: "

#: iw/account_gui.py:54
msgid "_Confirm:"
msgstr "Подтвер_дите: "

#: iw/account_gui.py:97
msgid "Caps Lock is on."
msgstr "Включен Caps Lock."

#: iw/account_gui.py:107 iw/account_gui.py:115 iw/account_gui.py:122
#: iw/account_gui.py:142 textw/userauth_text.py:74
msgid "Error with Password"
msgstr "Ошибка пароля"

#: iw/account_gui.py:108
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"Для продолжения необходимо ввести пароль root и подтвердить его, повторив "
"ввод."

#: iw/account_gui.py:116
msgid "The passwords you entered were different.  Please try again."
msgstr "Введенные вами пароли не совпадают.  Повторите ввод."

#: iw/account_gui.py:123
msgid "The root password must be at least six characters long."
msgstr "Пароль root должен содержать не менее 6 символов."

#: iw/account_gui.py:132 textw/userauth_text.py:83
msgid "Weak Password"
msgstr "Слабый пароль"

#: iw/account_gui.py:133
#, python-format
msgid "You have provided a weak password: %s"
msgstr ""

#: iw/account_gui.py:135
#, fuzzy
msgid "Use Anyway"
msgstr "Имя пользователя"

#: iw/account_gui.py:143 textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""
"Запрошенный пароль содержит символы, не являющиеся кодами ASCII. В паролях "
"такие символы не допускаются."

#: iw/autopart_type.py:100
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""
"Нет разделов, размер которых можно изменить. Только физические разделы с "
"выборочными файловыми системами могут быть изменены."

#: iw/autopart_type.py:123
msgid "Resize FileSystem Error"
msgstr "Ошибка изменения размера ФС"

#: iw/autopart_type.py:124 iw/autopart_type.py:133
#, python-format
msgid "%s: %s"
msgstr "%s: %s"

#: iw/autopart_type.py:132
msgid "Resize Device Error"
msgstr "Ошибка изменения размера устройства"

#: iw/autopart_type.py:207
msgid ""
"Do you really want to boot from a disk which is not used for installation?"
msgstr ""
"Вы действительно хотите загрузиться с диска, который не используется для "
"установки?"

#: iw/autopart_type.py:284
msgid "Invalid Initiator Name"
msgstr "Неверное имя инициатора"

#: iw/autopart_type.py:285
msgid "You must provide an initiator name."
msgstr "Вы должны указать имя инициатора."

#: iw/autopart_type.py:311
msgid "Error with Data"
msgstr "Ошибка данных"

#: iw/autopart_type.py:469 iw/autopart_type.py:470
msgid "Rescanning disks"
msgstr "Повторное сканирование дисков"

#: iw/autopart_type.py:524 textw/partition_text.py:63
msgid "Use entire drive"
msgstr "Использовать весь диск"

#: iw/autopart_type.py:525 textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr "Заменить существующую систему Linux"

#: iw/autopart_type.py:526
msgid "Shrink current system"
msgstr "Сжать существующую систему"

#: iw/autopart_type.py:527 textw/partition_text.py:65
msgid "Use free space"
msgstr "Использовать свободное пространство"

#: iw/autopart_type.py:528
msgid "Create custom layout"
msgstr "Создать собственное разбиение"

#: iw/blpasswidget.py:47
msgid "_Use a boot loader password"
msgstr "_Установить пароль загрузчика"

#: iw/blpasswidget.py:48
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"Пароль начального загрузчика защищает от изменения пользователями "
"передаваемых ядру опций."

#: iw/blpasswidget.py:79
msgid "Change _password"
msgstr "Изменить _пароль"

#: iw/blpasswidget.py:102
msgid "Enter Boot Loader Password"
msgstr "Введите пароль загрузчика"

#: iw/blpasswidget.py:108
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""
"Введите пароль загрузчика и подтвердите его.  (Учтите, что раскладка "
"клавиатуры в BIOS может отличаться от используемой сейчас.)"

#: iw/blpasswidget.py:115
msgid "_Password:"
msgstr "_Пароль:"

#: iw/blpasswidget.py:121
msgid "Con_firm:"
msgstr "Подтвердите:"

#: iw/blpasswidget.py:142
msgid "Passwords don't match"
msgstr "Пароли не совпадают"

#: iw/blpasswidget.py:143
msgid "Passwords do not match"
msgstr "Пароли не совпадают"

#: iw/blpasswidget.py:152
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"Ваш пароль загрузчика содержит менее 6 символов. Рекомендуется использование "
"более длинного пароля.\n"
"\n"
"Оставить текущий пароль? "

#: iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr "Настройка загрузчика"

#: iw/bootloader_main_gui.py:179 iw/bootloader_main_gui.py:184
#: iw/bootloader_main_gui.py:221
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "_Установите загрузчик на /dev/%s...."

#: iw/bootloader_main_gui.py:227
msgid "_Change device"
msgstr "_Изменить устройство"

#: iw/congrats_gui.py:33
msgid "Congratulations"
msgstr "Поздравляем"

#: iw/congrats_gui.py:74 textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""
"Поздравляем, установка %s завершена.\n"
"\n"

#: iw/congrats_gui.py:77 textw/complete_text.py:42
msgid "Shutdown"
msgstr "Выключение"

#: iw/congrats_gui.py:79 textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr ""
"Чтобы начать использовать установленную систему, выключите её.\n"
"\n"

#: iw/congrats_gui.py:81 textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr "Чтобы начать использовать установленную систему, перезагрузитесь.\n"

#: iw/congrats_gui.py:86 textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""
"Обратите внимание, что уже могут быть доступны обновления, которые нужны для "
"корректной работы вашей системы. Рекомендуется установить эти обновления "
"после перезагрузки."

#: iw/congrats_gui.py:90 textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""
"Поздравляем, установка %s завершена.\n"
"\n"
"Чтобы начать использовать установленную систему, перезагрузитесь.Обратите "
"внимание, что уже могут быть доступны обновления, которые нужны для "
"корректной работы вашей системы. Рекомендуется установить эти обновления "
"после перезагрузки."

#: iw/examine_gui.py:37
msgid "Upgrade Examine"
msgstr "Проверка возможности обновления"

#: iw/examine_gui.py:59
#, python-format
msgid "_Install %s"
msgstr "_Установить %s"

#: iw/examine_gui.py:61
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""
"Выберите этот параметр для установки новой системы. Существующее ПО и данные "
"могут быть переписаны в зависимости от выбранных параметров."

#: iw/examine_gui.py:65
msgid "_Upgrade an existing installation"
msgstr "_Обновить существующую установку"

#: iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""
"Выберите этот пункт, если вы хотите обновить существующую систему %s. "
"Существующие на ваших дисках данные будут сохранены."

#: iw/examine_gui.py:123
msgid "The following installed system will be upgraded:"
msgstr "Следующая установленная система будет обновлена:"

#: iw/examine_gui.py:136
msgid "Unknown Linux system"
msgstr "Неизвестная система Linux"

#: iw/language_gui.py:33
msgid "Language Selection"
msgstr "Выбор языка"

#: iw/language_gui.py:81 loader/lang.c:377
msgid "What language would you like to use during the installation process?"
msgstr "Какой язык должен использоваться в процессе установки?"

#: iw/lvm_dialog_gui.py:121 iw/lvm_dialog_gui.py:167 iw/lvm_dialog_gui.py:179
#: iw/lvm_dialog_gui.py:218 iw/lvm_dialog_gui.py:295 iw/lvm_dialog_gui.py:662
#: iw/lvm_dialog_gui.py:682
msgid "Not enough space"
msgstr "Недостаточно места"

#: iw/lvm_dialog_gui.py:122
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""
"Размер физического экстента не может быть изменен, потому что в противном "
"случае пространство, требуемое уже определенными логическими томами, будет "
"больше, чем доступное свободное пространство."

#: iw/lvm_dialog_gui.py:131
msgid "Confirm Physical Extent Change"
msgstr "Подтвердите изменение физического экстента"

#: iw/lvm_dialog_gui.py:132
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"Это изменение размера физического экстента потребует, чтобы текущие "
"запрошенные размеры логических томов были округлены до значения, кратного "
"физическому экстенту.\n"
"\n"
"Это изменение вступит в силу немедленно."

#: iw/lvm_dialog_gui.py:141 iw/lvm_dialog_gui.py:201
msgid "C_ontinue"
msgstr "Продолжить"

#: iw/lvm_dialog_gui.py:168
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the "
"volume group."
msgstr ""
"Размер физического экстента не может быть изменен, потому что выбранная "
"величина (%10.2f Мб) больше, чем наименьший физический том (%10.2f Мб) в "
"группе томов."

#: iw/lvm_dialog_gui.py:180
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is too large compared to the size of the smallest physical volume "
"(%10.2f MB) in the volume group."
msgstr ""
"Размер физического экстента не может быть изменен, потому что выбранная "
"величина (%10.2f Мб) слишком велика по сравнению с размером наименьшего "
"физического тома (%10.2f Мб) в группе томов."

#: iw/lvm_dialog_gui.py:194
msgid "Too small"
msgstr "Слишком мало места"

#: iw/lvm_dialog_gui.py:195
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"Это изменение значения физического экстента потребует значительного "
"дополнительного пространства на одном или более физических томов в группе "
"томов."

#: iw/lvm_dialog_gui.py:219
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""
"Размер физического экстента не может быть изменен, потому что полученный "
"размер логического тома (%10.2f Мб) будет меньше, чем один или более уже "
"определенных логических томов."

#: iw/lvm_dialog_gui.py:296
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""
"Нельзя удалить этот физический том, потому что размер группы томов будет "
"слишком мал для уже определенных логических томов."

#: iw/lvm_dialog_gui.py:395
msgid "Make Logical Volume"
msgstr "Создание логического тома"

#: iw/lvm_dialog_gui.py:397
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "Изменение логического тома: %s"

#: iw/lvm_dialog_gui.py:428 iw/raid_dialog_gui.py:378
msgid "_File System Type:"
msgstr "Тип ФС:"

#: iw/lvm_dialog_gui.py:434
msgid "_Logical Volume Name:"
msgstr "_Имя логического тома:"

#: iw/lvm_dialog_gui.py:443 iw/partition_dialog_gui.py:411
msgid "_Size (MB):"
msgstr "Размер (Мб):"

#: iw/lvm_dialog_gui.py:449
#, python-format
msgid "(Max size is %s MB)"
msgstr "(Макс. размер %s Мб)"

#: iw/lvm_dialog_gui.py:453 iw/partition_dialog_gui.py:463
#: iw/partition_ui_helpers_gui.py:338 iw/raid_dialog_gui.py:350
msgid "_Encrypt"
msgstr "_Шифрование"

#: iw/lvm_dialog_gui.py:462 iw/raid_dialog_gui.py:388
msgid "Original File System Type:"
msgstr "Текущий тип файловой системы:"

#: iw/lvm_dialog_gui.py:466 iw/partition_gui.py:776 iw/partition_gui.py:930
#: iw/raid_dialog_gui.py:393
msgid "Unknown"
msgstr "Неизвестный"

#: iw/lvm_dialog_gui.py:470 iw/partition_dialog_gui.py:401
#: iw/raid_dialog_gui.py:399
msgid "Original File System Label:"
msgstr "Текущий ярлык файловой системы:"

#: iw/lvm_dialog_gui.py:475
msgid "Logical Volume Name:"
msgstr "Имя логического тома:"

#: iw/lvm_dialog_gui.py:479
msgid "Size (MB):"
msgstr "Размер (Мб):"

#: iw/lvm_dialog_gui.py:506 iw/partition_dialog_gui.py:356
#: iw/raid_dialog_gui.py:369
msgid "_Mount Point:"
msgstr "Точка монтирования:"

#: iw/lvm_dialog_gui.py:589
msgid "Illegal Logical Volume Name"
msgstr "Неверное имя логического тома"

#: iw/lvm_dialog_gui.py:604
msgid "Illegal logical volume name"
msgstr "Неверное имя логического тома"

#: iw/lvm_dialog_gui.py:605
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr "Имя логического тома \"%s\" уже используется. Выберите другое имя."

#: iw/lvm_dialog_gui.py:633
msgid "Mount point in use"
msgstr "Точка монтирования используется"

#: iw/lvm_dialog_gui.py:634
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""
"Точка монтирования \"%s\" уже используется, выберите другую точку "
"монтирования."

#: iw/lvm_dialog_gui.py:649
msgid "Illegal size"
msgstr "недопустимый размер"

#: iw/lvm_dialog_gui.py:650
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "Введенный размер не является действительным числом больше 0."

#: iw/lvm_dialog_gui.py:663
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). To increase this limit you can create more Physical "
"Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr ""
"Запрошенный размер (%10.2f МБ) больше, чем максимальный размер логического "
"тома (%10.2f МБ). Чтобы увеличить это ограничение, вы можете создать "
"дополнительные Физические тома (PV) в неиспользуемом пространстве диска и "
"добавить их в Группу томов (VG)."

#: iw/lvm_dialog_gui.py:683
#, python-format
msgid ""
"The logical volumes you have configured require %d MB, but the volume group "
"only has %d MB.  Please either make the volume group larger or make the "
"logical volume(s) smaller."
msgstr ""

#: iw/lvm_dialog_gui.py:769
msgid "No free slots"
msgstr "Нет свободных слотов"

#: iw/lvm_dialog_gui.py:780
msgid "No free space"
msgstr "Недостаточно свободного места"

#: iw/lvm_dialog_gui.py:781
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""
"В группе томов нет места для создания новых логических томов. Чтобы добавить "
"логический том, вам необходимо уменьшить размер одного или нескольких уже "
"существующих логических томов"

#: iw/lvm_dialog_gui.py:813
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "Вы действительно хотите удалить логический том \"%s\"?"

#: iw/lvm_dialog_gui.py:926
msgid "Invalid Volume Group Name"
msgstr "Неверное имя группы томов"

#: iw/lvm_dialog_gui.py:935
msgid "Name in use"
msgstr "Имя используется"

#: iw/lvm_dialog_gui.py:936
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr "Имя группы томов \"%s\" уже используется, выберите другое имя."

#: iw/lvm_dialog_gui.py:1179
msgid "Not enough physical volumes"
msgstr "Недостаточно физических томов"

#: iw/lvm_dialog_gui.py:1180
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""
"Как минимум один неиспользуемый раздел физического тома необходим для "
"создания группы томов LVM.\n"
"\n"
"Создайте раздел или массив RAID типа \"физический том (LVM)\" и после этого "
"снова выберите параметр \"LVM\"."

#: iw/lvm_dialog_gui.py:1191
msgid "Make LVM Volume Group"
msgstr "Создать группу томов LVM"

#: iw/lvm_dialog_gui.py:1194
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "Правка группы томов LVM: %s"

#: iw/lvm_dialog_gui.py:1196
msgid "Edit LVM Volume Group"
msgstr "Правка группы томов LVM"

#: iw/lvm_dialog_gui.py:1212
msgid "_Volume Group Name:"
msgstr "_Имя группы томов:"

#: iw/lvm_dialog_gui.py:1220
msgid "Volume Group Name:"
msgstr "Имя группы томов:"

#: iw/lvm_dialog_gui.py:1228
msgid "_Physical Extent:"
msgstr "_Физический экстент:"

#: iw/lvm_dialog_gui.py:1243
msgid "Physical Volumes to _Use:"
msgstr "Физические тома:"

#: iw/lvm_dialog_gui.py:1249
msgid "Used Space:"
msgstr "Использовано:"

#: iw/lvm_dialog_gui.py:1266
msgid "Free Space:"
msgstr "Свободно:"

#: iw/lvm_dialog_gui.py:1284
msgid "Total Space:"
msgstr "Всего места:"

#: iw/lvm_dialog_gui.py:1322
msgid "Logical Volume Name"
msgstr "Имя логического тома"

#: iw/lvm_dialog_gui.py:1325 iw/partition_gui.py:362
#: iw/upgrade_swap_gui.py:140 textw/upgrade_text.py:118
msgid "Mount Point"
msgstr "Точка монтирования"

#: iw/lvm_dialog_gui.py:1328 iw/partition_gui.py:367
msgid "Size (MB)"
msgstr "Размер (Мб)"

#: iw/lvm_dialog_gui.py:1342 iw/osbootwidget.py:95
msgid "_Add"
msgstr "_Добавить"

#: iw/lvm_dialog_gui.py:1345 iw/osbootwidget.py:99 iw/partition_gui.py:1335
msgid "_Edit"
msgstr "_Изменить"

#: iw/lvm_dialog_gui.py:1360
msgid "_Logical Volumes"
msgstr "_Логические тома"

#: iw/netconfig_dialog.py:192 textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""
"Произошла ошибка преобразования значения для \"%s\":\n"
"%s"

#: iw/netconfig_dialog.py:194 iw/netconfig_dialog.py:203
#: textw/netconfig_text.py:35 textw/netconfig_text.py:40
msgid "Error With Data"
msgstr "Ошибка данных"

#: iw/netconfig_dialog.py:202
#, python-format
msgid "A value is required for the field %s."
msgstr "Требуется значение для поля %s."

#: iw/netconfig_dialog.py:212
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "Произошла ошибка при включении сетевого интерфейса %s."

#: iw/netconfig_dialog.py:214
msgid "Error Configuring Network"
msgstr "Ошибка при настройке сети"

#: iw/netconfig_dialog.py:240
#, fuzzy
msgid "Dynamic IP Address"
msgstr "Динамический IP"

#: iw/netconfig_dialog.py:241
#, fuzzy, python-format
msgid "Sending request for IP address information for %s"
msgstr "Посылается запрос конфигурации IP для %s..."

#: iw/netconfig_dialog.py:256 iw/netconfig_dialog.py:259
#: textw/netconfig_text.py:215 textw/netconfig_text.py:218
msgid "IP Address"
msgstr "Адрес IP"

#: iw/netconfig_dialog.py:266 textw/netconfig_text.py:225
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "Префикс IPv4 CIDR должен быть в диапазоне от 0 до 32."

#: iw/netconfig_dialog.py:267 iw/netconfig_dialog.py:273
#: iw/netconfig_dialog.py:281 iw/netconfig_dialog.py:284
#: textw/netconfig_text.py:226 textw/netconfig_text.py:232
#: textw/netconfig_text.py:240
msgid "IPv4 Network Mask"
msgstr "Маска сети IPv4"

#: iw/netconfig_dialog.py:294 textw/netconfig_text.py:253
msgid "Gateway"
msgstr "Шлюз"

#: iw/netconfig_dialog.py:304 textw/netconfig_text.py:263
msgid "Nameserver"
msgstr "Сервер имен"

#: iw/netconfig_dialog.py:313
msgid "Error configuring network device:"
msgstr "Ошибка при конфигурации сетевого устройства:"

#: iw/network_gui.py:66 iw/network_gui.py:72
msgid "Error with Hostname"
msgstr "Ошибка имени узла"

#: iw/network_gui.py:67
msgid "You must enter a valid hostname for this computer."
msgstr "Необходимо указать имя узла для этого компьютера."

#: iw/network_gui.py:73
#, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""
"Имя компьютера \"%s\" является недействительным по следующим причинам:\n"
"\n"
"%s"

#: iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr "Список операционных систем загрузчика"

#: iw/osbootwidget.py:66
msgid "Default"
msgstr "По умолчанию"

#: iw/osbootwidget.py:66 iw/partition_gui.py:361
msgid "Label"
msgstr "Ярлык"

#: iw/osbootwidget.py:66 iw/partition_gui.py:360
msgid "Device"
msgstr "Устройство"

#: iw/osbootwidget.py:129
msgid "Image"
msgstr "Образ"

#: iw/osbootwidget.py:136
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""
"Укажите метку, которая будет отображаться в меню начального загрузчика. "
"Устройство (жесткий диск или номер раздела) - это устройство, с которого "
"будет загружаться данная система."

#: iw/osbootwidget.py:144
msgid "_Label"
msgstr "_Ярлык"

#: iw/osbootwidget.py:152
msgid "_Device"
msgstr "Устройство"

#: iw/osbootwidget.py:182
msgid "Default Boot _Target"
msgstr "Образ загрузки по умолчанию"

#: iw/osbootwidget.py:211
msgid "You must specify a label for the entry"
msgstr "Необходимо указать ярлык для записи"

#: iw/osbootwidget.py:220
msgid "Boot label contains illegal characters"
msgstr "Ярлык загрузки содержит недопустимые символы."

#: iw/osbootwidget.py:244
msgid "Duplicate Label"
msgstr "Одинаковые ярлыки"

#: iw/osbootwidget.py:245
msgid "This label is already in use for another boot entry."
msgstr "Этот ярлык уже используется для другой записи."

#: iw/osbootwidget.py:258
msgid "Duplicate Device"
msgstr "Устройство используется"

#: iw/osbootwidget.py:259
msgid "This device is already being used for another boot entry."
msgstr "Это устройство уже используется для другой записи."

#: iw/osbootwidget.py:321
msgid "Cannot Delete"
msgstr "Невозможно удалить"

#: iw/osbootwidget.py:322
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""
"Эту цель загрузки нельзя удалить, потому что это устанавливаемая вами "
"система %s."

#: iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr "Доп. опции размера"

#: iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr "Фикс. размер"

#: iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr "Заполнить все пространство до (Мб):"

#: iw/partition_dialog_gui.py:74
msgid "Fill to maximum _allowable size"
msgstr "Занять все доступное пространство"

#: iw/partition_dialog_gui.py:326
msgid "Add Partition"
msgstr "Добавить раздел"

#: iw/partition_dialog_gui.py:328
#, python-format
msgid "Edit Partition: %s"
msgstr "Правка раздела: %s"

#: iw/partition_dialog_gui.py:365
msgid "File System _Type:"
msgstr "_Тип ФС:"

#: iw/partition_dialog_gui.py:381
msgid "Allowable _Drives:"
msgstr "Доступные устройства:"

#: iw/partition_dialog_gui.py:450
msgid "Force to be a _primary partition"
msgstr "Сделать первичным разделом"

#: iw/partition_gui.py:319
#, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr "Диск %s (%-0.f Мб) (Модель: %s)"

#: iw/partition_gui.py:363
msgid "Type"
msgstr "Тип"

#: iw/partition_gui.py:366
msgid "Format"
msgstr "Формат"

#: iw/partition_gui.py:404
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"Точка монт./\n"
"RAID/Том"

#: iw/partition_gui.py:406
msgid ""
"Size\n"
"(MB)"
msgstr ""
"Размер\n"
"(Мб)"

#: iw/partition_gui.py:518
msgid "Partitioning"
msgstr "Разбиение"

#: iw/partition_gui.py:608
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr "Выбранная схема разбиения вызвала следующие критические ошибки."

#: iw/partition_gui.py:610
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr "Эти ошибки должны быть исправлены до продолжения установки %s."

#: iw/partition_gui.py:616
msgid "Partitioning Errors"
msgstr "Ошибки разбиения"

#: iw/partition_gui.py:623
msgid "The partitioning scheme you requested generated the following warnings."
msgstr "Выбранная схема разбиения вызвала следующие предупреждения."

#: iw/partition_gui.py:625
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "Продолжить с текущей схемой разбиения?"

#: iw/partition_gui.py:630
msgid "Partitioning Warnings"
msgstr "Предупреждения разбиения"

#: iw/partition_gui.py:651
msgid "Format Warnings"
msgstr "Предупреждения форматирования"

#: iw/partition_gui.py:656
msgid "_Format"
msgstr "_Форматировать"

#: iw/partition_gui.py:683
msgid "LVM Volume Groups"
msgstr "Группы томов LVM"

#: iw/partition_gui.py:730
msgid "RAID Devices"
msgstr "Устройства RAID"

#: iw/partition_gui.py:797 loader/hdinstall.c:222
msgid "Hard Drives"
msgstr "Жесткие диски"

#: iw/partition_gui.py:898
msgid "Extended"
msgstr "Расширенный"

#: iw/partition_gui.py:900
msgid "software RAID"
msgstr "программный RAID"

#: iw/partition_gui.py:932
msgid "Free"
msgstr "Своб."

#: iw/partition_gui.py:1022 storage/partitioning.py:180
#: storage/partitioning.py:220
msgid "Error Partitioning"
msgstr "Ошибка разбиения"

#: iw/partition_gui.py:1023
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "Невозможно распределить запрошенные разделы: %s"

#: iw/partition_gui.py:1032
#, python-format
msgid "Warning: %s."
msgstr "Внимание: %s"

#: iw/partition_gui.py:1062 iw/partition_gui.py:1069
msgid "Unable To Edit"
msgstr "Правка невозможна"

#: iw/partition_gui.py:1063
msgid "You must select a device to edit"
msgstr "Необходимо указать устройство для правки."

#: iw/partition_gui.py:1070
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"Невозможно изменить это устройство:\n"
"\n"
"%s"

#: iw/partition_gui.py:1177 iw/partition_gui.py:1189
msgid "Not supported"
msgstr "Не поддерживается"

#: iw/partition_gui.py:1178
msgid "LVM is NOT supported on this platform."
msgstr "LVM НЕ поддерживается на этой платформе"

#: iw/partition_gui.py:1190
msgid "Software RAID is NOT supported on this platform."
msgstr "Программный RAID не поддерживается на этой платформе"

#: iw/partition_gui.py:1197
msgid "No RAID minor device numbers available"
msgstr "Нет доступных младших устройств (minor devices) RAID."

#: iw/partition_gui.py:1198
msgid ""
"A software RAID device cannot be created because all of the available RAID "
"minor device numbers have been used."
msgstr ""
"Устройство программного RAID не может быть создано, потому что все доступные "
"младшие номера устройств RAID уже заняты."

#: iw/partition_gui.py:1210
msgid "RAID Options"
msgstr "Параметры RAID"

#: iw/partition_gui.py:1225
#, fuzzy, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation."
msgstr ""
"Программный RAID позволяет объединить несколько дисков в одно большое "
"устройство RAID.  Устройство RAID может иметь большую производительность и "
"надежность по сравнению с отдельным устройством. Для получения "
"дополнительной информации об устройствах RAID обратитесь к документации по \n"
"%s.\n"
"\n"
"Число разделов RAID, готовых к использованию - %s.\n"
"\n"

#: iw/partition_gui.py:1235
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""
"Для использования программного RAID необходимо создать как минимум два "
"раздела типа 'software RAID'. После этого вы сможете создать и смонтировать "
"устройство RAID.\n"
"\n"

#: iw/partition_gui.py:1241
msgid "What do you want to do now?"
msgstr "Что бы вы хотели сделать сейчас?"

#: iw/partition_gui.py:1250
msgid "Create a software RAID _partition."
msgstr "Создать раздел программного RAID."

#: iw/partition_gui.py:1253
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr "Создать устройство RAID (по умолчанию /dev/md%s)."

#: iw/partition_gui.py:1257
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr "Клонировать диск для создания устройства RAID [default=/dev/md%s]"

#: iw/partition_gui.py:1299
msgid "Couldn't Create Drive Clone Editor"
msgstr "Не удается создать редактор клонирования дисков"

#: iw/partition_gui.py:1300
msgid "The drive clone editor could not be created for some reason."
msgstr ""
"Редактор клонирования дисков не может быть запущен по некоторым причинам."

#: iw/partition_gui.py:1334
msgid "Ne_w"
msgstr "Создать"

#: iw/partition_gui.py:1337
msgid "Re_set"
msgstr "_Отменить"

#: iw/partition_gui.py:1338
msgid "R_AID"
msgstr "R_AID"

#: iw/partition_gui.py:1339
msgid "_LVM"
msgstr "_LVM"

#: iw/partition_gui.py:1380
msgid "Hide RAID device/LVM Volume _Group members"
msgstr "Скрыть устройства RAID/участников группы LVM"

#: iw/partition_ui_helpers_gui.py:108 iw/partition_ui_helpers_gui.py:130
#: iw/partition_ui_helpers_gui.py:132
msgid "<Not Applicable>"
msgstr "<Не применимо>"

#: iw/partition_ui_helpers_gui.py:319
msgid "_Format as:"
msgstr "_Форматировать как:"

#: iw/partition_ui_helpers_gui.py:341
msgid "Mi_grate filesystem to:"
msgstr "_Перенести файловую систему на:"

#: iw/partition_ui_helpers_gui.py:371
msgid "_Resize"
msgstr "И_зменить размер"

#: iw/partition_ui_helpers_gui.py:435
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""
"Разделы типа '%s' должны быть объединены в один диск. Это производится путем "
"выбора диска в списке 'Доступные диски'."

#: iw/progress_gui.py:37
msgid "Installing Packages"
msgstr "Установка пакетов"

#: iw/raid_dialog_gui.py:317
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"Для создания устройства RAID необходимы как минимум два неиспользуемых "
"раздела RAID.\n"
"Создайте хотя бы два раздела типа \"программный RAID\" и выберите параметр "
"\"RAID\" снова."

#: iw/raid_dialog_gui.py:331 iw/raid_dialog_gui.py:768
msgid "Make RAID Device"
msgstr "Создать устройство RAID"

#: iw/raid_dialog_gui.py:334
#, python-format
msgid "Edit RAID Device: %s"
msgstr "Изменить устройство RAID: %s"

#: iw/raid_dialog_gui.py:336
msgid "Edit RAID Device"
msgstr "Изменить устройство RAID"

#: iw/raid_dialog_gui.py:407
msgid "RAID _Device:"
msgstr "Устройство RAID:"

#: iw/raid_dialog_gui.py:425
msgid "RAID _Level:"
msgstr "Уровень RAID:"

#: iw/raid_dialog_gui.py:467
msgid "_RAID Members:"
msgstr "Члены RAID:"

#: iw/raid_dialog_gui.py:484
msgid "Number of _spares:"
msgstr "Кол-во резервных:"

#: iw/raid_dialog_gui.py:494
msgid "_Format partition?"
msgstr "Форматировать раздел?"

#: iw/raid_dialog_gui.py:568
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""
"Диск-источник не имеет разделов, которые можно клонировать.  Для "
"клонирования диска необходимо определить разделы типа \"программный RAID\"."

#: iw/raid_dialog_gui.py:572 iw/raid_dialog_gui.py:578
#: iw/raid_dialog_gui.py:592 iw/raid_dialog_gui.py:605
msgid "Source Drive Error"
msgstr "Ошибка исходного диска"

#: iw/raid_dialog_gui.py:579
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""
"Диск-источник имеет разделы, которые не являются разделами типа 'программный "
"RAID'.\n"
"\n"
"Перед клонированием эти разделы должны быть удалены. "

#: iw/raid_dialog_gui.py:593
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""
"Выбранный исходный диск имеет разделы, которые не объединены в диск %s.\n"
"\n"
"Эти разделы должны быть удалены или ограничены на этот диск перед операцией "
"клонирования диска."

#: iw/raid_dialog_gui.py:606
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""
"Выбранный исходный диск содержит разделы программного RAID, которые являются "
"участниками активного устройства RAID.\n"
"\n"
"Эти разделы должны быть удалены перед операцией клонирования диска."

#: iw/raid_dialog_gui.py:620 iw/raid_dialog_gui.py:626
#: iw/raid_dialog_gui.py:642
msgid "Target Drive Error"
msgstr "Ошибка целевого диска"

#: iw/raid_dialog_gui.py:621
msgid "Please select the target drives for the clone operation."
msgstr "Укажите целевые устройства для операций клонирования."

#: iw/raid_dialog_gui.py:627
#, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr "Исходный диск /%s не может быть также выбран в качестве целевого."

#: iw/raid_dialog_gui.py:643
#, python-format
msgid ""
"The target drive %s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""
"Целевой диск %s содержит раздел, который не может быть удален по следующим "
"причинам:\n"
"\n"
"\"%s\"\n"
"\n"
"Этот раздел должен быть удален заранее."

#: iw/raid_dialog_gui.py:706
msgid "Please select a source drive."
msgstr "Укажите исходный диск."

#: iw/raid_dialog_gui.py:726
#, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""
"Диск %s сейчас будет клонирован на следующие диски:\n"
"\n"

#: iw/raid_dialog_gui.py:731
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""
"\n"
"\n"
"ВНИМАНИЕ! ВСЕ ДАННЫЕ НА ЦЕЛЕВЫХ ДИСКАХ БУДУТ УНИЧТОЖЕНЫ"

#: iw/raid_dialog_gui.py:734
msgid "Final Warning"
msgstr "Последнее предупреждение"

#: iw/raid_dialog_gui.py:736
msgid "Clone Drives"
msgstr "Клонирование дисков"

#: iw/raid_dialog_gui.py:745
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr "Сбой клонирования: произошла ошибка при очистке целевых дисков."

#: iw/raid_dialog_gui.py:778
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required "
"to setup RAID arrays.  This tool uses a source drive which has been prepared "
"with the desired partitioning layout, and clones this layout onto other "
"similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on "
"that drive only, and can only contain unused software RAID partitions.  "
"Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""
"Clone Drive Tool\n"
"\n"
"Эта утилита позволяет значительно упростить создание массива RAID. Она "
"позволяет взять исходный диск с уже подготовленными разделами и клонировать "
"его на другие устройства подобного размера. После этого будет создано "
"устройство RAID.\n"
" \n"
"ПРИМЕЧАНИЕ: На исходном диске допускаются разделы только типа \"программный "
"RAID\". Разделы другого типа не допускаются.\n"
"\n"
"ВСЕ ДАННЫЕ на диске(-ах) назначения будут уничтожены."

#: iw/raid_dialog_gui.py:798
msgid "Source Drive:"
msgstr "Исходный диск:"

#: iw/raid_dialog_gui.py:806
msgid "Target Drive(s):"
msgstr "Целевой диск (диски):"

#: iw/raid_dialog_gui.py:814
msgid "Drives"
msgstr "Устройства"

#: iw/task_gui.py:64
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""
"Невозможно прочитать метаданные пакета из хранилища. Возможная причина - "
"отсутствие каталога repodata.  Убедитесь, что дерево установки сформировано "
"правильно.\n"
"\n"
"%s"

#: iw/task_gui.py:162
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""
"Репозитарий %s уже добавлен, выберите другое имя репозитария и адрес URL."

#: iw/task_gui.py:175
msgid "Edit Repository"
msgstr "Изменить репозитарий"

#: iw/task_gui.py:238
msgid "Invalid Proxy URL"
msgstr "Неверный URL прокси"

#: iw/task_gui.py:239
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr "Вы должны указать HTTP. HTTPS или FTP URL этого прокси."

#: iw/task_gui.py:250 iw/task_gui.py:388
msgid "Invalid Repository URL"
msgstr "Неверный URL хранилища"

#: iw/task_gui.py:251 iw/task_gui.py:389
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr "Вы должны указать HTTP, HTTPS или FTP URL этого хранилища."

#: iw/task_gui.py:273 iw/task_gui.py:398
msgid "No Media Found"
msgstr "Носители не найдены"

#: iw/task_gui.py:274 iw/task_gui.py:399
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr "Не найдены установочные носители. Вставьте диск и повторите попытку."

#: iw/task_gui.py:293 iw/task_gui.py:415
msgid "Please enter an NFS server and path."
msgstr "Укажите сервер NFS и путь."

#: iw/task_gui.py:325
msgid "Invalid Repository Name"
msgstr "Неверное имя хранилища"

#: iw/task_gui.py:326
msgid "You must provide a repository name."
msgstr "Вы должны указать имя репозитария."

#: iw/task_gui.py:450 ui/addrepo.glade:10
msgid "Add Repository"
msgstr "Добавить хранилище"

#: iw/task_gui.py:455
msgid "No Software Repos Enabled"
msgstr "Не добавлены программные репозитории"

#: iw/task_gui.py:456
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""
"Для продолжения установки необходимо, чтобы был активен как минимум один "
"репозиторий."

#: iw/timezone_gui.py:63 textw/timezone_text.py:96
msgid "Time Zone Selection"
msgstr "Выбор часового пояса"

#: iw/upgrade_bootloader_gui.py:37 textw/upgrade_bootloader_text.py:136
msgid "Upgrade Boot Loader Configuration"
msgstr "Обновление настройки загрузчика"

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr "Обновить настройку загрузчика"

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr "Ваш текущий загрузчик будет обновлен"

#: iw/upgrade_bootloader_gui.py:128 textw/upgrade_bootloader_text.py:108
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""
"В результате изменений в системе конфигурация вашего загрузчика не может "
"быть автоматически обновлена."

#: iw/upgrade_bootloader_gui.py:131 textw/upgrade_bootloader_text.py:112
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""
"Программе установки не удается определить загрузчик, установленный на вашей "
"системе."

#: iw/upgrade_bootloader_gui.py:138 textw/upgrade_bootloader_text.py:121
#, python-format
msgid ""
"The installer has detected the %s boot loader currently installed on %s."
msgstr "Программа установки обнаружила загрузчик %s, установленный на %s."

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr "Это рекомендуемый параметр."

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr "Создать новую конфигурацию загрузчика"

#: iw/upgrade_bootloader_gui.py:149
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""
"Это позволит создать новую конфигурацию загрузчика. Выберите этот параметр, "
"если вы хотите изменить загрузчик."

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr "Пропустить обновление загрузчика"

#: iw/upgrade_bootloader_gui.py:157
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""
"Это позволит не изменять конфигурацию загрузчика. Выберите этот параметр, "
"если вы используете загрузчик, не входящий в дистрибутив."

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr "Что бы вы хотели сделать?"

#: iw/upgrade_migratefs_gui.py:38 textw/upgrade_text.py:36
msgid "Migrate File Systems"
msgstr "Преобразование файловых систем"

#: iw/upgrade_migratefs_gui.py:70
#, python-format
msgid ""
"This release of %s supports an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"Эта версия %s поддерживает обновленную файловую систему, которая обладает "
"преимуществами по сравнению с традиционно поставляемой с %s файловой "
"системой. Программа установки может выполнить миграцию отформатированных "
"разделов без потери данных.\n"
"\n"
"Какие из этих разделов вы хотите преобразовать?"

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr "Обновление раздела подкачки"

#: iw/upgrade_swap_gui.py:94 textw/upgrade_text.py:101
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""
"Последним ядрам (2.4 и новее) нужно значительно больше пространства "
"подкачки, чем более ранним ядрам, как минимум вдвое больше объема ОЗУ в "
"вашей системе.  Сейчас у вас выделено для подкачки %d МБ, но вы можете "
"создать дополнительное пространство подкачки на одной из ваших файловых "
"систем."

#: iw/upgrade_swap_gui.py:101
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"Установщик обнаружил %s Мб ОЗУ.\n"

#: iw/upgrade_swap_gui.py:113
msgid "I _want to create a swap file"
msgstr "Я хочу создать файл подкачки"

#: iw/upgrade_swap_gui.py:122
msgid "Select the _partition to put the swap file on:"
msgstr "Выберите раздел, где будет размещен файл подкачки:"

#: iw/upgrade_swap_gui.py:140 textw/upgrade_text.py:119
msgid "Partition"
msgstr "Раздел"

#: iw/upgrade_swap_gui.py:140
msgid "Free Space (MB)"
msgstr "Свободно (Мб)"

#: iw/upgrade_swap_gui.py:158
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""
"Рекомендуется, чтобы размер файла подкачки был не менее %d МБ. Введите "
"размер файла:"

#: iw/upgrade_swap_gui.py:173
msgid "Swap file _size (MB):"
msgstr "Размер файла _подкачки (Мб)"

#: iw/upgrade_swap_gui.py:183
msgid "I _don't want to create a swap file"
msgstr "Я не хочу создавать файл подкачки"

#: iw/upgrade_swap_gui.py:193
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""
"Настоятельно рекомендуется создать файл подкачки. В противном случае работа "
"установщика может быть аварийно прекращена. Вы уверены, что хотите "
"продолжить?"

#: iw/upgrade_swap_gui.py:201 textw/upgrade_text.py:188
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "Размер файла подкачки должен быть в диапазоне между 1 и 2000 Мб."

#: iw/upgrade_swap_gui.py:208 textw/upgrade_text.py:183
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr "Недостаточно места на устройстве, выбранном для раздела подкачки."

#: iw/welcome_gui.py:56 textw/welcome_text.py:36
msgid "Network Install Required"
msgstr "Требуется установка по сети"

#: iw/welcome_gui.py:57 textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""
"Источник вашей установки указывает на сетевой ресурс, но в системе не "
"обнаружено ни одной сетевой карты. Для того чтобы отменить установку по "
"сети, загрузите систему с полного DVD, полного набора CD дисков или не "
"передавайте параметр repo=, указывающий на сетевой ресурс."

#: iw/welcome_gui.py:67
msgid "E_xit Installer"
msgstr "П_рервать установку"

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr "Настройка загрузчика z/IPL"

#: iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr "В вашей системе будет установлен загрузчик z/IPL."

#: iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""
"Сейчас будет установлен начальный загрузчик z/IPL в вашей системе.\n"
"\n"
"Корневым разделом будет тот, который вы ранее задали при настройке "
"разделов.\n"
"\n"
"Для запуска машины будет использоваться устанавливаемое по умолчанию ядро.\n"
"\n"
"Для внесения изменений после установки достаточно изменить конфигурационный "
"файл /etc/zipl.conf.\n"
"\n"
"Сейчас вы можете задать дополнительные параметры ядра, которые могут "
"понадобиться для вашей машины или при вашей настройке."

#: iw/zipl_gui.py:90 textw/zipl_text.py:71
msgid "Kernel Parameters"
msgstr "Параметры ядра"

#: iw/zipl_gui.py:93 iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "Параметры Chandev"

#: loader/cdinstall.c:185 loader/cdinstall.c:206 loader/mediacheck.c:60
msgid "Media Check"
msgstr "Проверка носителя"

#: loader/cdinstall.c:185 loader/cdinstall.c:188 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/method.c:326
msgid "Test"
msgstr "Проверка"

#: loader/cdinstall.c:185 loader/cdinstall.c:189
msgid "Eject Disc"
msgstr "Извлечь диск"

#: loader/cdinstall.c:186
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""
"Нажмите \"%s\" для проверки находящегося в устройстве CD или \"%s\" для "
"извлечения CD и установки другого диска для тестирования."

#: loader/cdinstall.c:207
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""
"Если у вас есть для проверки дополнительные носители, вставьте следующий "
"диск и нажмите \"%s\". Тестирование каждого диска не требуется, но "
"настоятельно рекомендуется. Желательно протестировать каждый CD перед первым "
"использованием. После успешного тестирования не требуется тестировать их "
"повторно перед каждым использованием."

#: loader/cdinstall.c:229
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""
"Диск %s не был обнаружен в устройствах CD-ROM. Вставьте диск %s и нажмите %s "
"для продолжения."

#: loader/cdinstall.c:248
msgid "Disc Found"
msgstr "Диск обнаружен"

#: loader/cdinstall.c:249
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"Для начала проверки носителей перед установкой нажмите %s.\n"
"\n"
"Нажмите %s, чтобы пропустить проверку носителя и запустить установку."

#: loader/cdinstall.c:339
msgid "Scanning"
msgstr "Выполняется проверка"

#: loader/cdinstall.c:339
#, fuzzy, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr "Выполняется поиск установочных образов на устройстве %s"

#: loader/cdinstall.c:341
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr "Выполняется поиск установочных образов на устройстве %s"

#: loader/cdinstall.c:442
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""
"Диск %s не был обнаружен в устройствах CD-ROM. Вставьте диск %s и нажмите %s "
"для продолжения."

#: loader/cdinstall.c:451
msgid "Disc Not Found"
msgstr "Диск не обнаружен"

#: loader/cdinstall.c:452 loader/driverdisk.c:244 loader/driverdisk.c:275
#: loader/driverdisk.c:359 loader/driverselect.c:81 loader/driverselect.c:179
#: loader/driverselect.c:205 loader/hdinstall.c:222 loader/hdinstall.c:280
#: loader/kbd.c:138 loader/loader.c:483 loader/loader.c:503
#: loader/loader.c:1382 loader/loader.c:1413 loader/net.c:572 loader/net.c:943
#: loader/net.c:1856 loader/net.c:1878 loader/nfsinstall.c:91
#: loader/urls.c:292 storage/__init__.py:134 storage/devicetree.py:82
#: textw/constants_text.py:52
msgid "Back"
msgstr "Назад"

#: loader/cdinstall.c:518
msgid "Cannot find kickstart file on CDROM."
msgstr "Не удается найти файл kickstart на CDROM."

#: loader/copy.c:51 loader/method.c:280
#, c-format
msgid "Failed to read directory %s: %m"
msgstr "Не удалось прочитать каталог %s: %m"

#: loader/driverdisk.c:139
msgid "Loading"
msgstr "Загрузка"

#: loader/driverdisk.c:139
#, fuzzy
msgid "Reading driver disk"
msgstr "Чтение диска с драйверами..."

#: loader/driverdisk.c:238 loader/driverdisk.c:270
msgid "Driver Disk Source"
msgstr "Источник диска драйверов"

#: loader/driverdisk.c:239
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""
"В вашей системе имеется несколько устройств, в которые может быть вставлен "
"диск драйверов. Какое устройство использовать?"

#: loader/driverdisk.c:271
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""
"На данном устройстве находятся несколько разделов, которые могут содержать "
"образ диска с драйверами. Какой из разделов использовать?"

#: loader/driverdisk.c:306
msgid "Failed to mount partition."
msgstr "Ошибка монтирования раздела."

#: loader/driverdisk.c:311
msgid "Select driver disk image"
msgstr "Выберите образ диска с драйверами"

#: loader/driverdisk.c:312
msgid "Select the file which is your driver disk image."
msgstr "Укажите файл, который является образом диска с драйверами."

#: loader/driverdisk.c:341
msgid "Failed to load driver disk from file."
msgstr "Ошибка загрузки диска с драйверами из файла."

#: loader/driverdisk.c:353
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr "Вставьте диск драйверов в /dev/%s и нажмите \"OK\" для продолжения."

#: loader/driverdisk.c:359
msgid "Insert Driver Disk"
msgstr "Вставьте Диск драйверов"

#: loader/driverdisk.c:372
msgid "Failed to mount driver disk."
msgstr "Ошибка монтирования диска драйверов."

#: loader/driverdisk.c:380
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr "Диск драйверов не подходит для этого выпуска %s."

#: loader/driverdisk.c:443
msgid "Manually choose"
msgstr "Выбрать вручную"

#: loader/driverdisk.c:444
msgid "Load another disk"
msgstr "Загрузить другой диск"

#: loader/driverdisk.c:445
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""
"На диске драйверов не было найдено устройств данного типа.  Хотите ли вы "
"выбрать драйвер вручную, продолжить все равно или загрузить другой диск "
"драйверов?"

#: loader/driverdisk.c:483
msgid "Driver disk"
msgstr "Диск драйверов"

#: loader/driverdisk.c:484
msgid "Do you have a driver disk?"
msgstr "У вас есть диск с драйвером?"

#: loader/driverdisk.c:493
msgid "More Driver Disks?"
msgstr "Дополнительные диски"

#: loader/driverdisk.c:494
msgid "Do you wish to load any more driver disks?"
msgstr "Загрузить дополнительные диски драйверов?"

#: loader/driverdisk.c:539 loader/driverdisk.c:576 loader/hdinstall.c:375
#: loader/kickstart.c:132 loader/kickstart.c:142 loader/kickstart.c:185
#: loader/kickstart.c:554 loader/modules.c:335 loader/modules.c:347
#: loader/net.c:1577 loader/net.c:1598 loader/nfsinstall.c:335
#: loader/urlinstall.c:418 loader/urlinstall.c:427 loader/urlinstall.c:436
msgid "Kickstart Error"
msgstr "Ошибка Kickstart"

#: loader/driverdisk.c:540
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "Неизвестный источник драйвера диска kickstart: %s"

#: loader/driverdisk.c:577
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""
"Следующий некорректный аргумент был передан kickstart команде driver disk: %"
"s:%s"

#: loader/driverselect.c:67
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""
"Укажите все параметры, которые вы хотите передать модулю %s, разделив их "
"пробелами.  Если вы не знаете, какие параметры передать, пропустите этот "
"экран, нажав клавишу \"ОК\". Список доступных параметров можно узнать, нажав "
"клавишу F1."

#: loader/driverselect.c:91
msgid "Enter Module Parameters"
msgstr "Введите параметры модуля"

#: loader/driverselect.c:178
msgid "No drivers found"
msgstr "Драйверы не найдены"

#: loader/driverselect.c:178
msgid "Load driver disk"
msgstr "Загрузка диска драйверов"

#: loader/driverselect.c:179
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""
"Драйверы для загрузки вручную не были найдены. Использовать диск драйверов?"

#: loader/driverselect.c:197
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""
"Выберите из списка драйвер, который необходимо загрузить. Если драйвера нет "
"в списке и у вас есть диск драйверов, нажмите F2."

#: loader/driverselect.c:206
msgid "Specify optional module arguments"
msgstr "Указать дополнительные параметры модулей"

#: loader/driverselect.c:234
msgid "Select Device Driver to Load"
msgstr "Выберите драйвер устройства"

#: loader/hdinstall.c:135
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""
"Установочный образ на жестком диске не найден. Проверьте ваши образы и "
"попробуйте еще раз."

#: loader/hdinstall.c:223
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""
"В вашей системе не обнаружен ни один жесткий диск. Будете ли вы настраивать "
"дополнительные устройства?"

#: loader/hdinstall.c:237
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""
"Укажите раздел и каталог в этом разделе, который содержит установочные "
"образы для %s. Если необходимый диск отсутствует в списке, нажмите F2 для "
"настройки дополнительных устройств."

#: loader/hdinstall.c:264
msgid "Directory holding image:"
msgstr "Каталог, содержащий образ:"

#: loader/hdinstall.c:292
msgid "Select Partition"
msgstr "Выбор раздела"

#: loader/hdinstall.c:344
#, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "Похоже, что устройство %s не содержит установочный образ."

#: loader/hdinstall.c:376
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr "Недопустимый аргумент команды метода кикстарта %s: :%s"

#: loader/hdinstall.c:446 loader/hdinstall.c:502
msgid "Cannot find kickstart file on hard drive."
msgstr "Не удается найти файл кикстарта на жестком диске."

#: loader/hdinstall.c:489
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "Не удается найти жесткий диск, определенный в BIOS как %s"

#: loader/kbd.c:136
msgid "Keyboard Type"
msgstr "Тип клавиатуры"

#: loader/kbd.c:137
msgid "What type of keyboard do you have?"
msgstr "Укажите тип клавиатуры"

#: loader/kickstart.c:133
#, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "Ошибка открытия файла кикстарта %s: %m"

#: loader/kickstart.c:143
#, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "Ошибка чтения содержимого файла кикстарта %s: %m"

#: loader/kickstart.c:186
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "Ошибка в %s в строке %d файла kickstart %s"

#: loader/kickstart.c:285
msgid "Cannot find ks.cfg on removable media."
msgstr "Не удалось найти ks.cfg на загрузочной дискете."

#: loader/kickstart.c:380
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""
"Невозможно загрузить файл kickstart. Измените параметр kickstart или нажмите "
"Отмена для продолжения в интерактивном режиме."

#: loader/kickstart.c:389
msgid "Error downloading kickstart file"
msgstr "Ошибка загрузки файла kickstart"

#: loader/kickstart.c:555
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr "Неверный аргумент команды kickstart для выключения %s: :%s"

#: loader/lang.c:64
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "Добро пожаловать в %s для %s - Режим восстановления"

#: loader/lang.c:65 loader/loader.c:226
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""
"  <Tab>/<Alt-Tab> между элементами | <Space> выбор | <F12> следующий экран "

#: loader/lang.c:376
msgid "Choose a Language"
msgstr "Выберите язык"

#: loader/loader.c:127
msgid "Local CD/DVD"
msgstr "Локальный CD/DVD"

#: loader/loader.c:128
msgid "Hard drive"
msgstr "Жесткий диск"

#: loader/loader.c:129
msgid "NFS directory"
msgstr "Каталог NFS"

#: loader/loader.c:434 loader/loader.c:478
msgid "Update Disk Source"
msgstr "Источник диска обновлений"

#: loader/loader.c:435
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""
"В вашей системе имеется несколько устройств, каждое из которых может служить "
"источником диска обновлений.  Какое устройство использовать?"

#: loader/loader.c:479
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""
"На данном устройстве находятся несколько разделов, которые могут содержать "
"образ диска с драйверами. Какой из разделов использовать?"

#: loader/loader.c:497
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr "Вставьте диск с обновлениями в %s и нажмите \"OK\" для продолжения."

#: loader/loader.c:503
msgid "Updates Disk"
msgstr "Диск с обновлениями"

#: loader/loader.c:521
msgid "Failed to mount updates disk"
msgstr "Ошибка монтирования диска с обновлениями"

#: loader/loader.c:526
msgid "Updates"
msgstr "Обновления"

#: loader/loader.c:526
#, fuzzy
msgid "Reading anaconda updates"
msgstr "Чтение обновлений anaconda..."

#: loader/loader.c:562
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""
"Невозможно загрузить образ с обновлениями. Укажите другое расположение "
"обновлений или нажмите «Отмена» для продолжения без установки обновлений."

#: loader/loader.c:571
msgid "Error downloading updates image"
msgstr "Ошибка загрузки образа обновлений"

#: loader/loader.c:1200
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "Недостаточно ОЗУ для установки %s на этой машине."

#: loader/loader.c:1257
msgid "Media Detected"
msgstr "Обнаружен носитель"

#: loader/loader.c:1258
#, fuzzy
msgid "Found local installation media"
msgstr "Обнаружен локальный установочный носитель..."

#: loader/loader.c:1374
msgid "Rescue Method"
msgstr "Метод восстановления"

#: loader/loader.c:1375
msgid "Installation Method"
msgstr "Метод установки"

#: loader/loader.c:1377
msgid "What type of media contains the rescue image?"
msgstr "Какой носитель содержит спасательный образ?"

#: loader/loader.c:1379
msgid "What type of media contains the installation image?"
msgstr "Какой носитель содержит установочный образ?"

#: loader/loader.c:1412
msgid "No driver found"
msgstr "Драйвер не найден"

#: loader/loader.c:1412
msgid "Select driver"
msgstr "Выберите драйвер"

#: loader/loader.c:1413
msgid "Use a driver disk"
msgstr "Использовать диск драйверов"

#: loader/loader.c:1414
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""
"Не удается найти ни одно устройство, необходимое для этого типа установки.  "
"Хотите ли вы выбрать драйвер вручную или использовать диск драйверов?"

#: loader/loader.c:1634
msgid "The following devices have been found on your system."
msgstr "В вашей системе были найдены следующие устройства."

#: loader/loader.c:1636
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""
"Не загружен ни один драйвер устройств. Вы хотите загрузить какой-либо сейчас?"

#: loader/loader.c:1640
msgid "Devices"
msgstr "Устройства"

#: loader/loader.c:1641
msgid "Done"
msgstr "Готово"

#: loader/loader.c:1642
msgid "Add Device"
msgstr "Добавить устройство"

#: loader/loader.c:1854
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "Загрузчик уже выполнялся.  Запускается оболочка.\n"

#: loader/loader.c:2230
#, fuzzy, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr ""
"Запускается anaconda %s, режим восстановления %s, подождите, пожалуйста...\n"

#: loader/loader.c:2232
#, fuzzy, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr ""
"Запускается anaconda %s, программа установки %s, подождите, пожалуйста...\n"

#: loader/mediacheck.c:46
#, c-format
msgid "Unable to find install image %s"
msgstr "Не удается найти установочный образ %s"

#: loader/mediacheck.c:52
#, fuzzy, c-format
msgid "Checking \"%s\"."
msgstr "Проверка \"%s\"..."

#: loader/mediacheck.c:54
#, fuzzy, c-format
msgid "Checking media."
msgstr "Проверяется носитель..."

#: loader/mediacheck.c:86
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""
"Не удается прочитать контрольную сумму диска из первичного описания тома.   "
"Это скорее всего означает, что диск был создан без добавления контрольной "
"суммы."

#: loader/mediacheck.c:93
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""
"Образ, который только что был проверен, содержит ошибки. Возможная причина - "
"сбой при загрузке или поврежденный диск.  Если это возможно, протрите диск и "
"попробуйте еще раз. Если ошибка повторяется, не следует продолжать установку."

#: loader/mediacheck.c:101
msgid "Success"
msgstr "Успешно"

#: loader/mediacheck.c:102
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""
"Проверка образа завершена успешно. Стоит отметить, однако, что проверка "
"может пропустить некоторые ошибки."

#: loader/method.c:323
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"Вы хотите проверить контрольную сумму образа ISO:\n"
"\n"
"   %s?"

#: loader/method.c:326
msgid "Checksum Test"
msgstr "Проверка контрольной суммы"

#: loader/modules.c:336
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr "Неверный аргумент device команды kickstart  %s: :%s"

#: loader/modules.c:348
msgid "A module name must be specified for the kickstart device command."
msgstr "Команде kickstart должно быть передано имя модуля."

#: loader/net.c:111
msgid "Invalid Prefix"
msgstr "Неверный префикс"

#: loader/net.c:112
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""
"Значение префикса должно быть в диапазоне от 1 до 32 для сетей IPv4 или от 1 "
"до 128 для сетей IPv6."

#: loader/net.c:462 loader/net.c:518
msgid "Network Error"
msgstr "Ошибка сети"

#: loader/net.c:463 loader/net.c:519
msgid "There was an error configuring your network interface."
msgstr "Ошибка при конфигурации сетевого интерфейса."

#: loader/net.c:550 textw/netconfig_text.py:131
msgid "Enable IPv4 support"
msgstr "Включить поддержку IPv4"

#: loader/net.c:564
msgid "Enable IPv6 support"
msgstr "Включить поддержку IPv6"

#: loader/net.c:604
msgid "Configure TCP/IP"
msgstr "Настройка TCP/IP"

#: loader/net.c:661
msgid "Missing Protocol"
msgstr "Отсутствует протокол"

#: loader/net.c:662
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "Необходимо выбрать как минимум один протокол (IPv4 или IPv6)."

#: loader/net.c:669
msgid "IPv4 Needed for NFS"
msgstr "IPv4 необходим для NFS"

#: loader/net.c:670
msgid "NFS installation method requires IPv4 support."
msgstr "Метод установки по NFS требует наличия поддержки IPv4."

#: loader/net.c:769
msgid "IPv4 address:"
msgstr "Адрес IPv4:"

#: loader/net.c:781 loader/net.c:848 ui/netconfig.glade:261
#: ui/netconfig.glade:362
msgid "/"
msgstr "/"

#: loader/net.c:836
msgid "IPv6 address:"
msgstr "Адрес IPv6:"

#: loader/net.c:902 textw/netconfig_text.py:158
msgid "Gateway:"
msgstr "Шлюз:"

#: loader/net.c:910
msgid "Name Server:"
msgstr "Сервер имен:"

#: loader/net.c:949
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""
"Введите адресс IPv4 и/или IPv6 и префикс (адрес / префикс). Для IPv4 "
"допустимо использование маски с точками в качестве разделителей или префикса "
"в стиле CIDR. Поля для шлюза и сервера имен должны содержать корректные "
"адреса IPv4 или IPv6."

#: loader/net.c:969
msgid "Manual TCP/IP Configuration"
msgstr "Ручная настройка TCP/IP"

#: loader/net.c:1092 loader/net.c:1099
msgid "Missing Information"
msgstr "Недостаточно информации"

#: loader/net.c:1093
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr "Необходимо ввести корректный адрес IPv4 и маску сети или префикс CIDR."

#: loader/net.c:1100
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr "Необходимо ввести корректный адрес IPv6 и префикс CIDR."

#: loader/net.c:1578
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr "Неверный аргумент сетевой команды kickstart %s: :%s"

#: loader/net.c:1599
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr "В сетевой команде неверно указан bootproto %s"

#: loader/net.c:1676
msgid "Seconds:"
msgstr "Секунды:"

#: loader/net.c:1851
msgid "Networking Device"
msgstr "Сетевое устройство"

#: loader/net.c:1852
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""
"В вашей системе имеется несколько сетевых адаптеров. Какой из них должен "
"использоваться для установки?"

#: loader/net.c:1856
msgid "Identify"
msgstr "Идентификация"

#: loader/net.c:1865
msgid "You can identify the physical port for"
msgstr "Можно идентифицировать физический порт для "

#: loader/net.c:1867
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""
"с помощью мерцания светового индикатора на протяжении заданного периода "
"времени. Введите число в диапазоне 1-30, которое определит интервал в "
"секундах."

#: loader/net.c:1877
msgid "Identify NIC"
msgstr "Определите NIC"

#: loader/net.c:1890
msgid "Invalid Duration"
msgstr "Недопустимая длительность"

#: loader/net.c:1891
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr "Время должно лежать в диапазоне от 1 до 30 секунд."

#: loader/net.c:1903
#, fuzzy, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr "Световые индикаторы порта %s мигают на протяжении %d секунд..."

#: loader/net.c:2071 loader/net.c:2075
#, fuzzy, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr "Ожидание настройки %s менеджером NetworkManager...\n"

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "Имя сервера NFS:"

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "Каталог %s:"

#: loader/nfsinstall.c:83
#, c-format
msgid "Please enter the server name and path to your %s installation image."
msgstr "Введите имя сервера и путь к установочному образу %s."

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "Настройка NFS"

#: loader/nfsinstall.c:249
msgid "That directory could not be mounted from the server."
msgstr "Этот каталог не может быть смонтирован с сервера."

#: loader/nfsinstall.c:260
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr "Этот каталог не содержит установочный образ %s."

#: loader/nfsinstall.c:336
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr "Неверный аргумент NFS метода kickstart %s: %s"

#: loader/telnetd.c:89 loader/telnetd.c:128
msgid "Telnet"
msgstr "Telnet"

#: loader/telnetd.c:89
#, fuzzy
msgid "Waiting for telnet connection."
msgstr "Ожидание соединения telnet..."

#: loader/telnetd.c:128
#, fuzzy
msgid "Running anaconda via telnet."
msgstr "Запуск anaconda по telnet..."

#: loader/urlinstall.c:84
#, c-format
msgid "Unable to retrieve %s://%s%s."
msgstr "Не удается получить %s://%s%s."

#: loader/urlinstall.c:176
msgid "Unable to retrieve the install image."
msgstr "Не удается получить установочный образ."

#: loader/urlinstall.c:419
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr "Неверный аргумент URL метода kickstart %s: %s"

#: loader/urlinstall.c:428
msgid "Must supply a --url argument to Url kickstart method."
msgstr "Для Url метода необходимо передать параметр --url."

#: loader/urlinstall.c:437
#, c-format
msgid "Unknown Url method %s"
msgstr "Неизвестный метод URL %s"

#: loader/urls.c:235 loader/urls.c:237 loader/urls.c:240
msgid "Retrieving"
msgstr "Получение"

#: loader/urls.c:295
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr "Введите URL-ссылку к установочному образу %s на вашем сервере."

#: loader/urls.c:321
msgid "URL Setup"
msgstr "Настройка URL"

#: loader/urls.c:329
msgid "You must enter a URL."
msgstr "Вы должны указать URL."

#: loader/urls.c:335
msgid "URL must be either an ftp or http URL"
msgstr "URL-ссылка должна содержать адрес FTP или HTTP."

#: loader/urls.c:346
msgid "Unknown Host"
msgstr "Неизвестный сервер"

#: loader/urls.c:347
#, c-format
msgid "%s is not a valid hostname."
msgstr "%s не является допустимым именем сервера."

#: loader/windows.c:65
msgid "Loading SCSI driver"
msgstr "Загрузка драйвера SCSI"

#: loader/windows.c:66
#, fuzzy, c-format
msgid "Loading %s driver"
msgstr "Загружается драйвер %s..."

#: storage/__init__.py:85
msgid "Unknown Device"
msgstr "Неизвестное устройство"

#: storage/__init__.py:86
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""
"Указанный устройством %s источник установки не найден. Проверьте параметры и "
"повторите попытку."

#: storage/__init__.py:97
msgid "Installation cannot continue."
msgstr "Установка не может быть продолжена."

#: storage/__init__.py:98
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""
"Выбранная вами конфигурация хранилища уже активирована. Вы не сможете "
"вернуться к окну редактирования диска. Продолжить установку?"

#: storage/__init__.py:126
msgid "Encrypt device?"
msgstr "Зашифровать устройство?"

#: storage/__init__.py:127
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""
"Вы включили шифорование блочного устройства, но не указали пароль. Вернитесь "
"на предыдущий этап и укажите пароль, в противном случае шифрация будет "
"отключена."

#: storage/__init__.py:150
msgid "Writing storage configuration to disk"
msgstr "Запись информации о хранилище на диск"

#: storage/__init__.py:151
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""
"Выбранные вами параметры разбиения на разделы будут записаны на диск. Все "
"данные в удаленных и переформатированных разделах будут потеряны."

#: storage/__init__.py:156
msgid "Go _back"
msgstr "В_ернуться"

#: storage/__init__.py:157
msgid "_Write changes to disk"
msgstr "_Сохранить изменения на диск"

#: storage/__init__.py:272
msgid "Finding Devices"
msgstr "Поиск устройств"

#: storage/__init__.py:273
#, fuzzy
msgid "Finding storage devices"
msgstr "Идет поиск устройств хранения..."

#: storage/__init__.py:511
msgid "This partition is holding the data for the hard drive install."
msgstr "Этот раздел содержит данные для установки с жесткого диска."

#: storage/__init__.py:516
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "Невозможно удалить DASD, отформатированный как LDL"

#: storage/__init__.py:522
#, python-format
msgid "This device is part of the RAID device %s."
msgstr "Это устройство является частью устройства RAID %s."

#: storage/__init__.py:525
msgid "This device is part of a RAID device."
msgstr "Это устройство является частью устройства RAID."

#: storage/__init__.py:530
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "Это устройство является частью группы томов LVM '%s'."

#: storage/__init__.py:533
msgid "This device is part of a LVM volume group."
msgstr "Это устройство является частью группы томов LVM."

#: storage/__init__.py:549
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""
"Этот раздел является расширенным и содержит логические разделы, которые "
"невозможно удалить:\n"
"\n"

#: storage/__init__.py:817
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"Вы не указали корневой раздел (/), который необходим для продолжения "
"установки %s."

#: storage/__init__.py:822
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"Размер вашего корневого раздела меньше 250Мб, как правило, этого "
"недостаточно для установки %s."

#: storage/__init__.py:828
#, fuzzy, python-format
msgid ""
"Your / partition is less than %s MB which is lower than recommended for a "
"normal %s install."
msgstr ""
"Раздел / меньше %s мегабайт, что меньше рекомендуемого размера для "
"нормальной установки %s. "

#: storage/__init__.py:838
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""

#: storage/__init__.py:845
#, python-format
msgid ""
"Your %s partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""
"Раздел %s меньше %s мегабайт, что меньше рекомендуемого размера для "
"нормальной установки %s. "

#: storage/__init__.py:872
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""
"Установка на USB устройство.  Существует возможность получения нерабочей "
"системы."

#: storage/__init__.py:875
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""
"Установка на FireWire устройство.  Существует возможность получения "
"нерабочей системы."

#: storage/__init__.py:882
#, fuzzy
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""
"Вы не создали раздел подкачки. В большинстве случаев он может значительно "
"улучшить производительность системы."

#: storage/__init__.py:887
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"Вы не создали раздел подкачки. В большинстве случаев он может значительно "
"улучшить производительность системы."

#: storage/__init__.py:894
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: storage/__init__.py:898
#, fuzzy, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""
"Точка монтирования \"%s\" уже используется, выберите другую точку "
"монтирования."

#: storage/__init__.py:909
msgid "No Drives Found"
msgstr "Диски не найдены"

#: storage/__init__.py:910
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""
"Произошла ошибка: устройства для создания новых файловых систем не найдены. "
"Проверьте оборудование для обнаружения вероятной причины."

#: storage/__init__.py:1103 storage/__init__.py:1112
msgid "Dirty File Systems"
msgstr "Грязные ФС"

#: storage/__init__.py:1104
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"Не все файловые системы были правильно отключены. Загрузите вашу систему "
"Linux, дождитесь окончания проверки файловых систем и правильно завершите "
"работу, после чего повторите попытку обновления.\n"
"%s"

#: storage/__init__.py:1113
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"Не все файловые системы были правильно отключены. Смонтировать их все "
"равно?\n"
"%s"

#: storage/__init__.py:1560
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""
"Устройство подкачки:\n"
"\n"
"     %s\n"
"\n"
"из файла /etc/fstab в данный момент используется как раздел для приостанова "
"работы, это означает, что ваша система находится в \"спящем режиме\". Для "
"выполнения обновления полностью остановите работу вашей системы вместо "
"перевода ее в \"спящий режим\"."

#: storage/__init__.py:1568
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""
"Устройство подкачки:\n"
"\n"
"     %s\n"
"\n"
"из файла /etc/fstab в данный момент используется как раздел для приостанова "
"работы, это означает, что ваша система находится в \"спящем режиме\". Если "
"вы выполняете новую установку, обязательно укажите установщику "
"переформатировать все разделы подкачки."

#: storage/__init__.py:1582
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"Ошибка подключения устройства подкачки %s: %s\n"
"\n"
"Файл /etc/fstab обновляемого раздела не содержит описания рабочего "
"устройства подкачки.\n"
"\n"
"Нажмите ОК для выхода из программы установки."

#: storage/__init__.py:1588
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"Ошибка подключения устройства подкачки %s: %s\n"
"\n"
"Наиболее вероятная причина - раздел не был инициализирован.\n"
"\n"
"Нажмите ОК для выхода из программы установки."

#: storage/__init__.py:1646 storage/__init__.py:1656
msgid "Invalid mount point"
msgstr "Неверная точка монтирования"

#: storage/__init__.py:1647
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Произошла ошибка при попытке создания %s. Установка не может быть "
"продолжена.\n"
"\n"
"Нажмите <Enter> для выхода из программы установки."

#: storage/__init__.py:1657
#, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Произошла ошибка при попытке создания %s: %s. Установка не может быть "
"продолжена.\n"
"\n"
"Нажмите <Enter> для выхода из программы установки."

#: storage/__init__.py:1670 storage/__init__.py:1691
msgid "Unable to mount filesystem"
msgstr "Невозможно смонтировать файловую систему"

#: storage/__init__.py:1671
#, python-format
msgid ""
"An error occurred mounting device %s as %s.  You may continue installation, "
"but there may be problems."
msgstr ""
"Произошла ошибка при монтировании устройства %s как %s. Установка может быть "
"продолжена, но возможны проблемы."

#: storage/__init__.py:1692
#, python-format
msgid ""
"An error occurred mounting device %s as %s: %s. This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Произошла ошибка при попытке подключения %s к %s: %s. Установка не может "
"быть продолжена.\n"
"\n"
"Нажмите <Enter> для выхода из программы установки."

#: storage/devicelibs/lvm.py:312
#, python-format
msgid "vginfo failed for %s"
msgstr "сбой вызова vginfo для %s"

#: storage/devicelibs/lvm.py:341
#, python-format
msgid "lvs failed for %s"
msgstr "сбой вызова lvs для %s"

#: storage/devicetree.py:91
msgid "Confirm"
msgstr "Подтвердите"

#: storage/devicetree.py:92
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""
"Вы действительно хотите пропустить ввод пароля для устройства %s?\n"
"\n"
"Если вы пропустите этот шаг, содержимое устройства будет недоступно в "
"процессе установки."

#: storage/devicetree.py:142
#, fuzzy, python-format
msgid ""
"Error processing drive %s %-0.fMB (%s).\n"
"\n"
"Maybe it needs to be reinitialized.  YOU WILL LOSE ALL DATA ON THIS DRIVE IF "
"YOU REINITIALIZE IT!%s"
msgstr ""
"Ошибка обработки диска %s.\n"
"Возможно вам необходимо его переинициализировать, ЭТО ПРИВЕДЕТ К ПОТЕРЕ ВСЕХ "
"ДАННЫХ НА ДИСКЕ!"

#: storage/devicetree.py:148
msgid "_Ignore drive"
msgstr "_Пропустить диск"

#: storage/devicetree.py:149
msgid "_Re-initialize drive"
msgstr "_Реинициализировать диск"

#: storage/devicetree.py:169
#, fuzzy, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %s.  You can reinitialize all related PVs "
"(%s) which will erase the LVM metadata, or ignore which will preserve the "
"contents."
msgstr ""
"Ошибка обработки LVM.\n"
"Похоже, что данные LVM несогласованы.. (%s) make(s) up %s. Вы можете "
"повторно инициализировать все соответствующие PVs, тем самым удалив "
"метаданные LVM. Или проигнорируйте это, сохранив старое содержимое."

#: storage/devicetree.py:174
#, fuzzy
msgid "_Ignore"
msgstr "_Игнорировать все"

#: storage/devicetree.py:175
#, fuzzy
msgid "_Re-initialize"
msgstr "_Реинициализировать диск"

#: storage/devicetree.py:1634 storage/devicetree.py:1683
#, fuzzy
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "Это устройство является частью группы томов LVM."

#: storage/formats/fs.py:66
msgid "attr dict must include a type"
msgstr "словарь attr должен включать тип"

#: storage/formats/fs.py:113
msgid "filesystem configuration missing a type"
msgstr "в описании файловой системы отсутствует поле тип"

#: storage/formats/fs.py:343
msgid "Formatting"
msgstr "Форматирование"

#: storage/formats/fs.py:344
#, fuzzy, python-format
msgid "Creating filesystem on %s"
msgstr "Создание файловой системы на %s..."

#: storage/formats/fs.py:440
msgid "Resizing"
msgstr "Изменение размера"

#: storage/formats/fs.py:441
#, fuzzy, python-format
msgid "Resizing filesystem on %s"
msgstr "Изменение размера файловой системы на %s..."

#: storage/formats/fs.py:484
msgid "Checking"
msgstr "Проверка"

#: storage/formats/fs.py:485
#, fuzzy, python-format
msgid "Checking filesystem on %s"
msgstr "Проверка файловой системы на %s..."

#: storage/iscsi.py:82 storage/iscsi.py:83
msgid "Scanning iSCSI nodes"
msgstr "Сканирование узлов SCSI"

#: storage/iscsi.py:160 storage/iscsi.py:161
msgid "Initializing iSCSI initiator"
msgstr "Инициализируется инициатор iSCSI"

#: storage/iscsi.py:197
msgid "iSCSI not available"
msgstr "iSCSI не доступно"

#: storage/iscsi.py:199
msgid "No initiator name set"
msgstr "Не задано имя инициатора iSCSI"

#: storage/iscsi.py:213
msgid "No iSCSI nodes discovered"
msgstr "Не обнаружены узлы iSCSI"

#: storage/iscsi.py:216 storage/iscsi.py:217
msgid "Logging in to iSCSI nodes"
msgstr "Регистрация на узле iSCSI"

#: storage/iscsi.py:239
msgid "No new iSCSI nodes discovered"
msgstr "Не обнаружено новых узлов iSCSI"

#: storage/iscsi.py:242
msgid "Could not log in to any of the discovered nodes"
msgstr "Не могу зарегистрироваться ни на одном из обнаруженных узлов"

#: storage/partitioning.py:181
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""
"Недостаточно свободногое места для автоматического разбиения. Выберите "
"другой способ разбиения диска."

#: storage/partitioning.py:205
msgid "Warnings During Automatic Partitioning"
msgstr "Предостережения при автоматическом разбиении"

#: storage/partitioning.py:207
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"При автоматическом разбиении были следующие предостережения:\n"
"\n"
"%s"

#: storage/partitioning.py:219 storage/partitioning.py:239
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"Нажмите OК для выхода из программы установки."

#: storage/partitioning.py:221
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr ""
"Невозможно распределить запрошенные разделы: \n"
"\n"
"%s.%s"

#: storage/partitioning.py:241
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""
"\n"
"\n"
"Нажмите 'OK' для выбора другого варианта разбиения."

#: storage/partitioning.py:243
msgid "Automatic Partitioning Errors"
msgstr "Ошибки автоматического разбиения"

#: storage/partitioning.py:244
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %s"
msgstr ""
"Выбранное вами разбиение вызвало следующие ошибки:\n"
"\n"
"%s\n"
"\n"
"Возможная причина ошибки - недостаточное количество свободного места для "
"установки. %s"

#: storage/partitioning.py:255
msgid "Unrecoverable Error"
msgstr "Неустранимая ошибка"

#: storage/zfcp.py:49
msgid "You have not specified a device number or the number is invalid"
msgstr "Вы не задали номер устройства или номер указан неправильно"

#: storage/zfcp.py:51
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""
"Вы не задали worldwide port name устройства или имя указано неправильно."

#: storage/zfcp.py:53
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "Вы не задали FCP LUN устройства или номер указан неправильно."

#: textw/complete_text.py:32
msgid "Reboot"
msgstr "Перезагрузка"

#: textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "<Enter> для выхода"

#: textw/complete_text.py:62
msgid "Complete"
msgstr "Готово"

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "Тип клавиатуры"

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "Укажите модель клавиатуры, подключенной к этому компьютеру"

#: textw/netconfig_text.py:41
#, python-format
msgid "A value is required for the field %s"
msgstr "Требуется значение для поля %s"

#: textw/netconfig_text.py:86 ui/netconfig.glade:8
msgid "Enable network interface"
msgstr "Включить _Сетевой интерфейс"

#: textw/netconfig_text.py:89 ui/netconfig.glade:78
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""
"Для этого необходимо наличие активного сетевого соединения на протяжении "
"процесса установки. Выполните настройку сетевого интерфейса."

#: textw/netconfig_text.py:128
msgid "Use dynamic IP configuration (DHCP)"
msgstr "Использовать динамическую конфигурацию IP (DHCP)"

#: textw/netconfig_text.py:138
msgid "IPv4 Address:"
msgstr "Адрес IPv4:"

#: textw/netconfig_text.py:161
msgid "Nameserver:"
msgstr "Сервер имен:"

#: textw/netconfig_text.py:191
msgid "Missing Device"
msgstr "Отсутствующее устройство"

#: textw/netconfig_text.py:192
msgid "You must select a network device"
msgstr "Необходимо выбрать сетевое устройство"

#: textw/netconfig_text.py:243
msgid "IPv4 Network Mask "
msgstr "Маска сети IPv4"

#: textw/netconfig_text.py:266
msgid "Configuring Network Interfaces"
msgstr "Настройка сетевых интерфейсов"

#: textw/netconfig_text.py:266
#, fuzzy
msgid "Waiting for NetworkManager"
msgstr "Ожидание NetworkManager..."

#: textw/netconfig_text.py:272
msgid "Error configuring network device"
msgstr "Ошибка при конфигурации сетевого интерфейса"

#: textw/netconfig_text.py:272
#, python-format
msgid "Error configuring network device %s"
msgstr "Ошибка при конфигурации сетевого интерфейса %s"

#: textw/partition_text.py:58
msgid "Partitioning Type"
msgstr "Тип разбиения"

#: textw/partition_text.py:60
#, fuzzy
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""
"Установка требует создания разделов на вашем жестком диске.  Разбиение по "
"умолчанию является оптимальным для большинства пользователей. Укажите какое "
"пространство на каких дисках использовать для установки. Вы также можете "
"создать разделы вручную."

#: textw/partition_text.py:80
msgid "Which drive(s) do you want to use for this installation?"
msgstr "Какие устройства вы хотите использовать для этой установки?"

#: textw/partition_text.py:95
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Space>,<+>,<-> выбор   |   <F2> Добавить диск  |  <F12> след. экран"

#: textw/partition_text.py:173 ui/adddrive.glade:8
msgid "Advanced Storage Options"
msgstr "Дополнительные параметры накопителей"

#: textw/partition_text.py:174 ui/adddrive.glade:138
msgid "How would you like to modify your drive configuration?"
msgstr "Как следует выполнить модификацию конфигурации диска?"

#: textw/partition_text.py:203
msgid "Add FCP Device"
msgstr "Добавить FCP устройство"

#: textw/partition_text.py:204 ui/zfcp-config.glade:77
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""
"Машины zSeries могут получать доступ к стандартным SCSI устройствам через "
"Fibre Channel (FCP). Для каждого устройства необходимо указать 16-битный "
"номер устройства, 64-битный World Wide Port Number (WWPN) и 64-битный FCP "
"LUN."

#: textw/partition_text.py:223
#, fuzzy
msgid "Add FCoE SAN"
msgstr "Добавить _ZFCP LUN"

#: textw/partition_text.py:224
msgid ""
"Enter the device name for the NIC which is connected to the FCoE SAN. For "
"example \"eth0\"."
msgstr ""

#: textw/partition_text.py:225
#, fuzzy
msgid "NIC device name"
msgstr "Устройства не найдены"

#: textw/partition_text.py:232
#, fuzzy, python-format
msgid "%s is not a valid NIC device name."
msgstr "%s не является допустимым именем сервера."

#: textw/partition_text.py:248 ui/iscsi-config.glade:7
msgid "Configure iSCSI Parameters"
msgstr "Настройка параметров iSCSI"

#: textw/partition_text.py:249 ui/iscsi-config.glade:139
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""
"Чтобы использовать диски iSCSI, вы должны предоставить адрес устройства "
"iSCSI и имя инициатора iSCSI, которое было настроено для вашего узла."

#: textw/partition_text.py:250
msgid "Target IP Address"
msgstr "Целевой IP адрес"

#: textw/partition_text.py:251
msgid "iSCSI Initiator Name"
msgstr "Имя инициатора iSCSI"

#: textw/partition_text.py:252
msgid "CHAP username"
msgstr "Имя пользователя CHAP"

#: textw/partition_text.py:253
msgid "CHAP password"
msgstr "Пароль CHAP"

#: textw/partition_text.py:254
msgid "Reverse CHAP username"
msgstr "Обратное имя пользователя CHAP"

#: textw/partition_text.py:255
msgid "Reverse CHAP password"
msgstr "Обратный пароль CHAP"

#: textw/progress_text.py:46
msgid "Package Installation"
msgstr "Установка пакета"

#: textw/timezone_text.py:75
msgid "In which time zone are you located?"
msgstr "В каком часовом поясе вы находитесь?"

#: textw/timezone_text.py:93
msgid "System clock uses UTC"
msgstr "Системные часы используют UTC"

#: textw/upgrade_bootloader_text.py:116 textw/upgrade_bootloader_text.py:125
msgid "Update boot loader configuration"
msgstr "Обновить настройку загрузчика"

#: textw/upgrade_bootloader_text.py:128
msgid "Skip boot loader updating"
msgstr "Пропустить обновление загрузчика"

#: textw/upgrade_bootloader_text.py:130
msgid "Create new boot loader configuration"
msgstr "Создать новую конфигурацию загрузчика"

#: textw/upgrade_text.py:38
#, python-format
msgid ""
"This release of %s supports the an updated file system, which has several "
"benefits over the file system traditionally shipped in %s.  This "
"installation program can migrate formatted partitions without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"Эта версия %s поддерживает обновленную файловую систему, которая обладает "
"преимуществами по сравнению с традиционно поставляемой с %s файловой "
"системой. Возможно преобразовать разделы ext2 в ext3 без потери данных.\n"
"\n"
"Какие из этих разделов вы хотите преобразовать?"

#: textw/upgrade_text.py:119
msgid "Free Space"
msgstr "Свободное пространство"

#: textw/upgrade_text.py:137
msgid "RAM detected (MB):"
msgstr "Обнаружено ОЗУ (Мб):"

#: textw/upgrade_text.py:140
msgid "Suggested size (MB):"
msgstr "Прогнозируемый размер (Мб):"

#: textw/upgrade_text.py:143
msgid "Swap file size (MB):"
msgstr "Размер файла подкачки (Мб):"

#: textw/upgrade_text.py:151
msgid "Add Swap"
msgstr "Добавить пространство подкачки"

#: textw/upgrade_text.py:176
msgid "The value you entered is not a valid number."
msgstr "Введенное значение должно быть числовой величиной."

#: textw/upgrade_text.py:209
msgid "Reinstall System"
msgstr "Переустановка системы"

#: textw/upgrade_text.py:219
msgid "System to Upgrade"
msgstr "Система для обновления"

#: textw/upgrade_text.py:220
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""
"В вашей системе найдена одна или несколько установок Linux.\n"
"\n"
"Выберите одну из них для обновления, или выберите \"Переустановка системы\" "
"для установки новой системы."

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr "Пароль пользователя root"

#: textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""
"Выберите пароль пользователя root. Вы должны ввести его дважды для "
"исключения ошибки при вводе."

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "Пароль пользователя root должен содержать не менее 6 символов."

#: textw/userauth_text.py:84
#, fuzzy, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"Заданный пароль не является сильным: %s\n"
"\n"
"Продолжить работу с этим паролем?"

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"Добро пожаловать в %s!\n"
"\n"

#: textw/zipl_text.py:35
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""
"Начальный загрузчик z/IPL может быть установлен в системе после завершения "
"установки. Сейчас вы можете ввести дополнительные параметры ядра и chandev, "
"которые могут требоваться для вашей машины или для вашей настройки."

#: textw/zipl_text.py:67
msgid "z/IPL Configuration"
msgstr "Настройка z/IPL"

#: textw/zipl_text.py:75 textw/zipl_text.py:79
msgid "Chandev line "
msgstr "Строка chandev "

#: ui/GroupSelector.glade:8
msgid "window1"
msgstr ""

#: ui/GroupSelector.glade:203
msgid "_Optional packages"
msgstr "_Дополнительные пакеты"

#: ui/GroupSelector.glade:233
msgid "dialog1"
msgstr "dialog1"

#: ui/GroupSelector.glade:293
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""
"Не обязательна установка всех пакетов, связанных с этой группой. Тем не "
"менее, они могут обеспечить дополнительную функциональность. Выберите "
"пакеты, которые вы хотите установить."

#: ui/GroupSelector.glade:360
msgid "_Select"
msgstr "Вы_брать"

#: ui/GroupSelector.glade:369
msgid "_Select all optional packages"
msgstr "Вы_брать все дополнительные пакеты"

#: ui/GroupSelector.glade:378
msgid "_Deselect"
msgstr "О_тменить выбор"

#: ui/GroupSelector.glade:387
msgid "_Deselect all optional packages"
msgstr "О_тменить выбор всех дополнительных пакетов"

#: ui/account.glade:78
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Учетная запись root используется для администрирования системы. Введите "
"пароль пользователя root."

#: ui/account.glade:134
msgid "Root Password:"
msgstr "Пароль root:"

#: ui/account.glade:162
msgid "Confirm:"
msgstr "Подтвердите:"

#: ui/adddrive.glade:92
msgid "_Add drive"
msgstr "_Добавить диск"

#: ui/adddrive.glade:169
msgid "Add _iSCSI target"
msgstr "Добавить устройство _iSCSI"

#: ui/adddrive.glade:188
msgid "Add _ZFCP LUN"
msgstr "Добавить _ZFCP LUN"

#: ui/adddrive.glade:207
#, fuzzy
msgid "Add _FCoE SAN"
msgstr "Добавить _ZFCP LUN"

#: ui/adddrive.glade:228
msgid "Disable _dmraid device"
msgstr "_Отключить устройство dmraid"

#: ui/addrepo.glade:82
msgid ""
"Please provide the configuration information for this software repository."
msgstr "Укажите настройки для этого репозитория с приложениями."

#: ui/addrepo.glade:116
msgid "<b>Repository _name:</b>"
msgstr "<b>_Имя хранилища:</b>"

#: ui/addrepo.glade:166
msgid "<b>Repository _type:</b>"
msgstr "<b>_Тип репозитория:</b>"

#: ui/addrepo.glade:194
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Жесткий диск"

#: ui/addrepo.glade:243
msgid "Repository _URL"
msgstr "_URL репозитария"

#: ui/addrepo.glade:296
msgid "URL is a _mirror list"
msgstr "URL содержит список _зеркал"

#: ui/addrepo.glade:316
msgid "Configure _proxy"
msgstr "Настройка п_рокси"

#: ui/addrepo.glade:344
msgid "Proxy U_RL (host:port)"
msgstr "_Адрес прокси (узел:порт)"

#: ui/addrepo.glade:373
msgid "Proxy u_sername"
msgstr "_Имя пользователя прокси"

#: ui/addrepo.glade:402
msgid "Proxy pass_word"
msgstr "_Пароль на прокси"

#: ui/addrepo.glade:640
msgid "_Server"
msgstr "_Сервер"

#: ui/addrepo.glade:669
msgid "_Path"
msgstr "П_уть"

#: ui/addrepo.glade:698
msgid "_Options"
msgstr "_Параметры"

#: ui/addrepo.glade:855
msgid "_Partition"
msgstr "_Раздел"

#: ui/addrepo.glade:883
msgid "_Directory"
msgstr "_Каталог"

#: ui/addrepo.glade:926
msgid "Select A Directory"
msgstr "Выберите каталог"

#: ui/anaconda.glade:253
msgid "_Next"
msgstr "_Далее"

#: ui/anaconda.glade:327
msgid "Reboo_t"
msgstr "_Перезагрузка"

#: ui/autopart.glade:33
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target. You can also choose to create your own custom "
"layout."
msgstr ""
"Установка требует создания разделов на вашем жестком диске.  Разбиение по "
"умолчанию является оптимальным для большинства пользователей. Укажите какое "
"пространство на каких дисках использовать для установки. Вы также можете "
"создать разделы вручную."

#: ui/autopart.glade:86
msgid "_Encrypt system"
msgstr "За_шифровать систему"

#: ui/autopart.glade:103
msgid "_Select the drive(s) to use for this installation."
msgstr "_Укажите устройства для этой установки:"

#: ui/autopart.glade:216
msgid "_Advanced storage configuration"
msgstr "_Дополнительная настройка накопителей"

#: ui/autopart.glade:271
msgid "What drive would you like to _boot this installation from?"
msgstr "С какого _диска вы хотите запустить установку?"

#: ui/autopart.glade:323
msgid "Re_view and modify partitioning layout"
msgstr "_Просмотр и изменение разбиения разделов"

#: ui/autopart.glade:344
#, fuzzy
msgid "Volume to Shrink"
msgstr "Изменение размера тома"

#: ui/autopart.glade:430
msgid "_Shrink"
msgstr ""

#: ui/autopart.glade:475
#, fuzzy
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""
"Укажите раздел, размер которого вы хотите увеличить для размещения установки?"

#: ui/autopart.glade:520
#, fuzzy
msgid "<b>Shrink _target (in MB):</b>"
msgstr "<b>Изменить _размер диска (в Мб):</b>"

#: ui/blwhere.glade:8
msgid "Boot loader device"
msgstr "Загрузчик"

#: ui/blwhere.glade:77
msgid "Where would you like to install the boot loader for your system?"
msgstr "Куда вы хотите установить загрузчик?"

#: ui/blwhere.glade:102
msgid "MBR"
msgstr "MBR"

#: ui/blwhere.glade:120
msgid "/boot"
msgstr "/boot"

#: ui/blwhere.glade:167
msgid "First BIOS drive:"
msgstr "Первое устройство BIOS:"

#: ui/blwhere.glade:194
msgid "Second BIOS drive:"
msgstr "Второе устройство BIOS:"

#: ui/blwhere.glade:221
msgid "Third BIOS drive:"
msgstr "Третье устройство BIOS:"

#: ui/blwhere.glade:248
msgid "Fourth BIOS drive:"
msgstr "Четвертое устройство BIOS:"

#: ui/blwhere.glade:341
msgid "BIOS Drive Order"
msgstr "Порядок дисков BIOS"

#: ui/detailed-dialog.glade:63
msgid "Info"
msgstr "Инфо"

#: ui/detailed-dialog.glade:133
msgid "_Details"
msgstr "_Подробности"

#: ui/instkey.glade:58
msgid "Please enter your %(instkey)s."
msgstr "Введите ваш %(instkey)s."

#: ui/instkey.glade:102
msgid "%(instkey)s:"
msgstr "%(instkey)s:"

#: ui/iscsi-config.glade:93
msgid "_Add target"
msgstr "_Добавить устройство"

#: ui/iscsi-config.glade:174
msgid "<b>_Target IP Address:</b>"
msgstr "<b>_Целевой IP адрес:</b>"

#: ui/iscsi-config.glade:226
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr "<b>_Имя инициатора iSCSI:</b>"

#: ui/iscsi-config.glade:278
msgid "<b>CHAP _Username:</b>"
msgstr "<b>_Имя пользователя CHAP:</b>"

#: ui/iscsi-config.glade:330
msgid "<b>CHAP _Password:</b>"
msgstr "<b>_Пароль CHAP:</b>"

#: ui/iscsi-config.glade:382
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "<b>Обратное и_мя пользователя CHAP:</b>"

#: ui/iscsi-config.glade:434
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "<b>Обратный па_роль CHAP:</b>"

#: ui/lukspassphrase.glade:8
msgid "Enter passphrase for encrypted partition"
msgstr "Введите пароль для зашифрованного раздела"

#: ui/lukspassphrase.glade:100
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""
"Выберите пароль для зашифрованного раздела. Ввод пароля будет будет запрошен "
"при каждой загрузке системы."

#: ui/lukspassphrase.glade:141
msgid "Enter passphrase:"
msgstr "Введите пароль:"

#: ui/lukspassphrase.glade:169
msgid "Confirm passphrase:"
msgstr "Подтвердите пароль:"

#: ui/lukspassphrase.glade:248
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""
"Используйте этот пароль для всех зашифрованных устройств, чтобы облегчить "
"процесс загрузки"

#: ui/lukspassphrase.glade:368
#, fuzzy
msgid "label1"
msgstr "Ярлык"

#: ui/netconfig.glade:109
msgid "<b>_Interface:</b>"
msgstr "<b>_Интерфейс:</b>"

#: ui/netconfig.glade:155
msgid "Use _dynamic IP configuration (DHCP)"
msgstr "Использовать _Динамическую конфигурацию IP (DHCP)"

#: ui/netconfig.glade:174
msgid "Enable IPv_4 support"
msgstr "Включить поддержку IPv_4"

#: ui/netconfig.glade:192
msgid "Enable IPv_6 support"
msgstr "Включить поддержку IPv_6"

#: ui/netconfig.glade:217
msgid "<b>IPv4 Address:</b>"
msgstr "<b>Адрес IPv4:</b>"

#: ui/netconfig.glade:318
msgid "<b>IPv6 Address:</b>"
msgstr "<b>Адрес IPv6:</b>"

#: ui/netconfig.glade:419
msgid "<b>Gateway:</b>"
msgstr "<b>Шлюз:</b>"

#: ui/netconfig.glade:476
msgid "<b>Nameserver:</b>"
msgstr "<b>Сервер имен:</b>"

#: ui/network.glade:78
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""
"Присвойте этому компьютеру имя узла, которое будет использоваться для его "
"идентификации в сети."

#: ui/network.glade:134
msgid "Hostname:"
msgstr "Имя узла:"

#: ui/tasksel.glade:35
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""
"Установка по умолчанию %s включает набор программ, подходящих для решения "
"обычных задач работы с Интернетом.  Поддержку каких дополнительных задач вы "
"хотели бы включить?"

#: ui/tasksel.glade:110
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""
"Вы можете продолжить настройку списка программ сейчас, или выполнить ее "
"после установки при помощи утилиты по управлению программным обеспечением."

#: ui/tasksel.glade:141
msgid "Customize _later"
msgstr "Настроить _позже"

#: ui/tasksel.glade:159
msgid "_Customize now"
msgstr "Настроить _сейчас"

#: ui/tasksel.glade:200
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""
"Укажите любые дополнительные репозитарии, которые вы хотели бы использовать "
"для установки ПО."

#: ui/tasksel.glade:304
msgid "_Add additional software repositories"
msgstr "_Добавить дополнительные хранилища пакетов"

#: ui/tasksel.glade:378
msgid "_Modify repository"
msgstr "_Изменить хранилище"

#: ui/zfcp-config.glade:7
msgid "Add FCP device"
msgstr "Добавить FCP устройство"

#: ui/zfcp-config.glade:112
msgid "<b>Device number:</b>"
msgstr "<b>Номер устройства:</b>"

#: ui/zfcp-config.glade:162
msgid "<b>WWPN:</b>"
msgstr "<b>WWPN:</b>"

#: ui/zfcp-config.glade:212
msgid "<b>FCP LUN:</b>"
msgstr "<b>FCP LUN:</b>"

#~ msgid "Fatal Error"
#~ msgstr "Фатальная ошибка"

#~ msgid ""
#~ "You do not have enough RAM to install %s on this machine.\n"
#~ "\n"
#~ "Press <return> to reboot your system.\n"
#~ msgstr ""
#~ "Недостаточно ОЗУ для установки %s на этой машине. \n"
#~ "\n"
#~ "Нажмите <Ввод> для перезагрузки.\n"

#~ msgid ""
#~ "Your bug could not be filed due to the following error when communicating "
#~ "with bugzilla:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Отчет об ошибке не был отправлен вследствие проблемы при обращении к "
#~ "Bugzilla:\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "Your bug could not be filed due to bad information in the bug fields.  "
#~ "This is most likely an error in anaconda:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Отчет об ошибке не был отправлен вследствие проблем в полях отчета. "
#~ "Вероятно, причиной является ошибка Anaconda:\n"
#~ "\n"
#~ "%s"

#~ msgid "Unable To File Bug"
#~ msgstr "Не удается отправить отчет об ошибке"

#~ msgid "Bug Filing Not Supported"
#~ msgstr "Создание отчетов об ошибках не поддерживается"

#~ msgid ""
#~ "Your distribution does not provide a supported bug filing system, so you "
#~ "cannot save your exception this way."
#~ msgstr ""
#~ "Ваш дистрибутив не включает в свой состав систему создания отчетов, "
#~ "поэтому вы не сможете сохранить информацию таким образом."

#~ msgid "Invalid Bug Information"
#~ msgstr "Неверная информация об ошибке"

#~ msgid ""
#~ "Please provide a valid username, password, and short bug description."
#~ msgstr ""
#~ "Укажите корректное имя пользователя, пароль и краткое описание проблемы."

#~ msgid "Unable To Login"
#~ msgstr "Авторизация не успешна"

#~ msgid ""
#~ "There was an error logging into %s using the provided username and "
#~ "password."
#~ msgstr ""
#~ "Ошибка при журналировании в %s с использованием указанных имени "
#~ "пользователя и пароля."

#~ msgid "Bug Created"
#~ msgstr "Отчет об ошибке создан"

#~ msgid ""
#~ "A new bug has been created with your traceback attached. Please add "
#~ "additional information such as what you were doing when you encountered "
#~ "the bug, screenshots, and whatever else is appropriate to the following "
#~ "bug:\n"
#~ "\n"
#~ "%s/%s"
#~ msgstr ""
#~ "Создан новый отчет об ошибке. Отладочная информация присоединена. "
#~ "Добавьте дополнительные сведения, например, опишите, что именно вы делали "
#~ "когда возникла ошибка, присоедините снимки экрана и т.д.\n"
#~ "\n"
#~ "%s/%s"

#~ msgid "Bug Updated"
#~ msgstr "Отчет об ошибке обновлен"

#~ msgid ""
#~ "A bug with your information already exists.  Your account has been added "
#~ "to the CC list and your traceback added as a comment.  Please add "
#~ "additional descriptive information to the following bug:\n"
#~ "\n"
#~ "%s/%s"
#~ msgstr ""
#~ "Отчет об ошибке уже существует. Ваша учетная запись добавлена в список "
#~ "CC, а отладочная информация добавлена в виде комментария. По желанию "
#~ "введите дополнительные сведения в отчет:\n"
#~ "\n"
#~ "%s/%s"

#~ msgid "Dump Written"
#~ msgstr "Дамп сохранен"

#~ msgid ""
#~ "Your system's state has been successfully written to the disk. The "
#~ "installer will now exit."
#~ msgstr ""
#~ "Состояние вашей системы было успешно сохранено на диск. Работа программы "
#~ "установки будет завершена."

#~ msgid "Dump Not Written"
#~ msgstr "Дамп не сохранен"

#~ msgid ""
#~ "Cannot save a bug report since there is no active networking device "
#~ "available."
#~ msgstr ""
#~ "Не удается сохранить отчет об ошибке, так как нет доступных сетевых "
#~ "устройств."

#~ msgid ""
#~ "Your system's state has been successfully written to the remote host.  "
#~ "The installer will now exit."
#~ msgstr ""
#~ "Состояние вашей системы было успешно сохранено на удаленный узел. "
#~ "Программа установки завершит работу."

#~ msgid "There was a problem writing the system state to the remote host."
#~ msgstr "Возникла проблема при записи состояния системы на удаленный узел."

#~ msgid "Bugzilla (%s)"
#~ msgstr "Bugzilla (%s)"

#~ msgid "Debug"
#~ msgstr "Отладка"

#~ msgid "Exception Occurred"
#~ msgstr "Произошло исключение"

#~ msgid "Error: %s splits into %s but not like we expect"
#~ msgstr "Ошибка: %s раделяется на %s, чего не ожидалось"

#~ msgid "Could not get information for mount point /boot or /"
#~ msgstr "Невозможно получить информацию о точке монтирования /boot или /"

#~ msgid "Running..."
#~ msgstr "Выполнение..."

#~ msgid "Doing post-installation"
#~ msgstr "Завершение установки"

#~ msgid "None"
#~ msgstr "Ничего"

#~ msgid ""
#~ "The device %s is LDL formatted instead of CDL formatted.  LDL formatted "
#~ "DASDs are not supported for usage during an install of %s.  If you wish "
#~ "to use this disk for installation, it must be re-initialized causing the "
#~ "loss of ALL DATA on this drive.\n"
#~ "\n"
#~ "Would you like to reformat this DASD using CDL format?"
#~ msgstr ""
#~ "Устройство %s имеет формат LDL вместо формата CDL. DASD с форматом LDL не "
#~ "поддерживаются в процессе установки %s.  Если вы хотите использовать этот "
#~ "диск для установки, его необходимо инициализировать повторно с потерей "
#~ "ВСЕХ ДАННЫХ на этом диске.\n"
#~ "\n"
#~ "Вы действительно хотите отформатировать DASD с использованием формата CDL?"

#~ msgid ""
#~ "/dev/%s currently has a %s partition layout.  To use this drive for the "
#~ "installation of %s, it must be re-initialized, causing the loss of ALL "
#~ "DATA on this drive.\n"
#~ "\n"
#~ "Would you like to re-initialize this drive?"
#~ msgstr ""
#~ "Устройство /dev/%s уже имеет таблицу разделов типа %s.  Чтобы "
#~ "использовать этот диск для установки %s, его необходимо инициализировать "
#~ "повторно с потерей ВСЕХ ДАННЫХ на этом диске.\n"
#~ "\n"
#~ "Вы действительно хотите реинициализировать этот диск?"

#~ msgid "Initializing"
#~ msgstr "Инициализация"

#~ msgid "Please wait while formatting drive %s...\n"
#~ msgstr "Подождите, идет форматирование устройства %s...\n"

#~ msgid ""
#~ "The partition table on device %s (%s %-0.f MB) was unreadable.\n"
#~ "\n"
#~ "To create new partitions it must be initialized, causing the loss of ALL "
#~ "DATA on this drive.\n"
#~ "\n"
#~ "This operation will override any previous installation choices about "
#~ "which drives to ignore.\n"
#~ "\n"
#~ "Would you like to initialize this drive, erasing ALL DATA?"
#~ msgstr ""
#~ "Невозможно прочитать таблицу разделов на устройстве %s (%s %-0.f Мб).\n"
#~ "Для создания новых разделов устройство должно быть инициализировано. \n"
#~ "При этом ВСЕ ДАННЫЕ на этом устройстве будут утеряны. \n"
#~ "\n"
#~ "Эта операция отменит сделанный ранее выбор игнорируемых дисков.\n"
#~ "\n"
#~ "Вы действительно хотите инициализировать этот диск с удалением ВСЕХ "
#~ "ДАННЫХ?"

#~ msgid ""
#~ "The drive /dev/%s has more than 15 partitions on it.  The SCSI subsystem "
#~ "in the Linux kernel does not allow for more than 15 partitons at this "
#~ "time.  You will not be able to make changes to the partitioning of this "
#~ "disk or use any partitions beyond /dev/%s15 in %s"
#~ msgstr ""
#~ "Устройство /dev/%s содержит более 15 разделов.  Подсистема SCSI ядра "
#~ "Linux на данный момент не позволяет работать с более, чем 15 разделами.  "
#~ "Вы не сможете изменить конфигурацию данного диска и не сможете работать с "
#~ "разделами выше /dev/%s15 на %s"

#~ msgid "Save"
#~ msgstr "Сохранить"

#~ msgid "Save to local disk"
#~ msgstr "Сохранить на локальный диск"

#~ msgid "Send to bugzilla (%s)"
#~ msgstr "Отправить в Bugzilla (%s)"

#~ msgid "Send to remote server (scp)"
#~ msgstr "Отправить на удаленный сервер (scp)"

#~ msgid "Password"
#~ msgstr "Пароль"

#~ msgid "Bug Description"
#~ msgstr "Описание проблемы"

#~ msgid "Host (host:port)"
#~ msgstr "Узел (узел:порт)"

#~ msgid "Destination file"
#~ msgstr "Целевой файл"

#~ msgid "The passphrase must be at least %d characters long."
#~ msgstr "Пароль должен содержать не менее %d символов."

#~ msgid ""
#~ "You have choosen the upgrade for %s architecture, but the installed "
#~ "system is for %s architecture. \n"
#~ "\n"
#~ msgstr ""
#~ "Вы выбрали выполнение обновления для архитектуры %s, но устанавливаемая "
#~ "система предназначена для архитектуры %s.\n"
#~ "\n"

#~ msgid ""
#~ "Would you like to upgrade  the installed system to the %s architecture?"
#~ msgstr "Вы хотите обновить установленную систему до уровня архитектуры %s?"

#~ msgid "Giving up attempting to connect after %d tries!\n"
#~ msgstr "Прекращение попыток подключения после %d неудачных!\n"

#~ msgid "%s Byte"
#~ msgstr "%s б"

#~ msgid "%s Bytes"
#~ msgstr "%s б"

#~ msgid "Processing"
#~ msgstr "Обработка"

#~ msgid "%s of %s packages completed"
#~ msgstr "%s из %s пакетов завершены"

#~ msgid "Starting install process.  This may take several minutes..."
#~ msgstr "Запуск процесса установки.  Это может занять несколько минут..."

#~ msgid "You cannot create more than %s logical volumes per volume group."
#~ msgstr "Вы не можете создать более чем %s логических томов в группе томов."

#~ msgid "Drive %s (Geom: %s/%s/%s) (Model: %s)"
#~ msgstr "Диск %s (Геом: %s/%s/%s) (Модель: %s)"

#~ msgid ""
#~ "To install the full set of supported packages included in your "
#~ "subscription, please enter your Installation Number"
#~ msgstr ""
#~ "Чтобы установить полный набор пакетов вашей подписки, введите Номер "
#~ "Установки"

#~ msgid ""
#~ "If you're unable to locate the Installation Number, consult http://www."
#~ "redhat.com/apps/support/in.html.\n"
#~ "\n"
#~ "If you skip:\n"
#~ "* You may not get access to the full set of packages included in your "
#~ "subscription.\n"
#~ "* It may result in an unsupported/uncertified installation of Red Hat "
#~ "Enterprise Linux.\n"
#~ "* You will not get software and security updates for packages not "
#~ "included in your subscription."
#~ msgstr ""
#~ "Если вы не можете найти Номер Установки, проконсультируйтесь с http://www."
#~ "redhat.com/apps/support/in.html.\n"
#~ "\n"
#~ "Если пропустить этот шаг:\n"
#~ "* Вы не сможете получить доступ к полному набору пакетов вашей подписки.\n"
#~ "* Установка Red Hat Enterprise Linux будет неподдерживаемой/"
#~ "несертифицированной.\n"
#~ "* Вы не получите обновлений пакетов, не включенных в вашу подписку."

#~ msgid "Filesystem error detected, cannot continue."
#~ msgstr "Обнаружена ошибка файловой системы. Продолжение невозможно."

#~ msgid "boot flag not available for this partition"
#~ msgstr "Признак загрузки недоступен для этого раздела"

#~ msgid "_Ignore drive(s)"
#~ msgstr "_Пропустить диск(и)"

#~ msgid "_Re-initialize drive(s)"
#~ msgstr "_Реинициализировать диск(и)"

#~ msgid "Your system will now be rebooted."
#~ msgstr "Ваша система сейчас будет перезагружена."

#~ msgid "Bug _description"
#~ msgstr "Описание о_шибки"

#~ msgid "Destination _file"
#~ msgstr "_Целевой файл"

#~ msgid ""
#~ "Local storage device\n"
#~ "Local disk\n"
#~ "Remote server (scp)"
#~ msgstr ""
#~ "Локальное устройство хранения\n"
#~ "Локальный диск\n"
#~ "Удаленный сервер (scp)"

#~ msgid "Please choose a destination for saving your traceback."
#~ msgstr "Выберите местоположение, где будет сохранена информация отладки."

#~ msgid "Select A File"
#~ msgstr "Выберите файл"

#~ msgid "_Host (host:port)"
#~ msgstr "_Узел (узел:порт)"

#~ msgid "_Password"
#~ msgstr "_Пароль"

#~ msgid "_User name"
#~ msgstr "_Имя пользователя"

#~ msgid "Install"
#~ msgstr "Установить"

#~ msgid "Install the live CD to your hard disk"
#~ msgstr "Установить с Live CD на жесткий диск"

#~ msgid "Install to Hard Drive"
#~ msgstr "Установить на жесткий диск"

#~ msgid "Afrikaans"
#~ msgstr "Африкаанс"

#~ msgid "Arabic"
#~ msgstr "Арабский"

#~ msgid "Assamese"
#~ msgstr "Ассамский"

#~ msgid "Bengali"
#~ msgstr "Бенгальский"

#~ msgid "Bengali(India)"
#~ msgstr "Бенгальский(Индия)"

#~ msgid "Bulgarian"
#~ msgstr "Болгарский"

#~ msgid "Catalan"
#~ msgstr "Каталонский"

#~ msgid "Chinese(Simplified)"
#~ msgstr "Китайский (упрощенный)"

#~ msgid "Chinese(Traditional)"
#~ msgstr "Китайский (традиционный)"

#~ msgid "Croatian"
#~ msgstr "Хорватский"

#~ msgid "Czech"
#~ msgstr "Чешский"

#~ msgid "Danish"
#~ msgstr "Датский"

#~ msgid "Dutch"
#~ msgstr "Голландский"

#~ msgid "English"
#~ msgstr "Английский"

#~ msgid "Estonian"
#~ msgstr "Эстонский"

#~ msgid "Finnish"
#~ msgstr "Финский"

#~ msgid "French"
#~ msgstr "Французский"

#~ msgid "German"
#~ msgstr "Немецкий"

#~ msgid "Greek"
#~ msgstr "Греческий"

#~ msgid "Gujarati"
#~ msgstr "Гуджарати"

#~ msgid "Hebrew"
#~ msgstr "Иврит"

#~ msgid "Hindi"
#~ msgstr "Хинди"

#~ msgid "Hungarian"
#~ msgstr "Венгерский"

#~ msgid "Icelandic"
#~ msgstr "Исландский"

#~ msgid "Iloko"
#~ msgstr "Илокский"

#~ msgid "Indonesian"
#~ msgstr "Индонезийский"

#~ msgid "Italian"
#~ msgstr "Итальянский"

#~ msgid "Japanese"
#~ msgstr "Японский"

#~ msgid "Kannada"
#~ msgstr "Каннада"

#~ msgid "Korean"
#~ msgstr "Корейский"

#~ msgid "Macedonian"
#~ msgstr "Македонский"

#~ msgid "Maithili"
#~ msgstr "Майтхили"

#~ msgid "Malay"
#~ msgstr "Малайский"

#~ msgid "Malayalam"
#~ msgstr "Малаялам"

#~ msgid "Marathi"
#~ msgstr "Маратхи"

#~ msgid "Nepali"
#~ msgstr "Непали"

#~ msgid "Norwegian(Bokmål)"
#~ msgstr "Норвежский (Букмол)"

#~ msgid "Northern Sotho"
#~ msgstr "Сото Северный"

#~ msgid "Oriya"
#~ msgstr "Ория"

#~ msgid "Persian"
#~ msgstr "Персидский"

#~ msgid "Polish"
#~ msgstr "Польский"

#~ msgid "Portuguese"
#~ msgstr "Португальский"

#~ msgid "Portuguese(Brazilian)"
#~ msgstr "Португальский(Бразильский)"

#~ msgid "Punjabi"
#~ msgstr "Панджаби"

#~ msgid "Romanian"
#~ msgstr "Румынский"

#~ msgid "Russian"
#~ msgstr "Русский"

#~ msgid "Serbian"
#~ msgstr "Сербский"

#~ msgid "Serbian(Latin)"
#~ msgstr "Сербский (лат)"

#~ msgid "Sinhala"
#~ msgstr "Сингальский"

#~ msgid "Slovak"
#~ msgstr "Словацкий"

#~ msgid "Slovenian"
#~ msgstr "Словенский"

#~ msgid "Spanish"
#~ msgstr "Испанский"

#~ msgid "Swedish"
#~ msgstr "Шведский"

#~ msgid "Tajik"
#~ msgstr "Таджикский"

#~ msgid "Tamil"
#~ msgstr "Тамильский"

#~ msgid "Telugu"
#~ msgstr "Телугу"

#~ msgid "Turkish"
#~ msgstr "Турецкий"

#~ msgid "Ukrainian"
#~ msgstr "Украинский"

#~ msgid "Vietnamese"
#~ msgstr "Вьетнамский"

#~ msgid "Welsh"
#~ msgstr "Валлийский"

#~ msgid "Zulu"
#~ msgstr "Зулусский"
